
<!DOCTYPE html>
<html lang="en">
<head>
	<title> Mean fitness at the mutation-selection balance </title>
	<meta name="description" content="Yoav Ram's blog on evolutionary theory and other scientific interests, and especially on the theoretical treatment of the evolution of the mutation rate" />
	<meta name="author" content="Yoav Ram">
	<meta name="keywords" content="evolution,mutation rate,mutation rates,Yoav Ram,yoavram,evolutionary theory,science,theoretical biology,mathematical biology,mathematics">
	<meta charset="UTF-8">
	<meta http-equiv="content-type" content="text/html; charset=UTF-8">
	<link rel="icon" type="image/ico" href="/static/favicon.ico">
	<link href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" title="Recent Articles" type="application/atom+xml">

	<!-- Le style -->
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link href='http://fonts.googleapis.com/css?family=Droid+Sans:400,700' rel='stylesheet' type='text/css'>
	<link href="/static/theme.css" rel="stylesheet">
	<link href="/static/bootstrap.min.css" rel="stylesheet">
	<link href="/static/glyphicons.css" rel="stylesheet">
	<link href="/static/bootstrap-responsive.min.css" rel="stylesheet">
		<link href="/static/jekyll.css" rel="stylesheet">

	<style type="text/css">
	[class^="icon-"] { background-image: url("/static/glyphicons-halflings.png"); } 
	[class*="icon-white"] { background-image: url("/static/glyphicons-halflings-white.png"); } 
	[class^="glyphicon-"] { background-image: url("/static/glyphicons.png"); } 
	</style>
	<link href="/static/prettify.css" rel="stylesheet">

	<!-- Le JS -->
	<script type="text/javascript" src="/static/jquery-1.7.2.min.js"></script>
	<script type="text/javascript" src="/static/bootstrap.min.js"></script>
	<script type="text/javascript">
	// http://henrywrites.wordpress.com/2012/09/03/jquery-string-format/
	String.prototype.format = String.prototype.f = function() {
	    var s = this,
	        i = arguments.length;

	    while (i--) {
	        s = s.replace(new RegExp('\\{' + i + '\\}', 'gm'), arguments[i]);
	    }
	    return s;
	};
</script>
	<!-- google code prettifier http://code.google.com/p/google-code-prettify/ -->
	<script type="text/javascript" src="/static/prettify.js"></script>

	



<!-- MathJax -->
<script type="text/x-mathjax-config">
<!-- Online Debugger: http://www.onemathematicalcat.org/MathJaxDocumentation/TeXSyntax.htm -->
<!-- Doc: http://docs.mathjax.org/en/latest/output.html -->`
  MathJax.Hub.Config({            
			"HTML-CSS": { availableFonts: ["STIX", "TeX"], 
						  linebreaks: { automatic: true }
			}
  });
</script>
<script type="text/javascript"
  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML">
</script>




<!-- ShareThis JS -->
<script type="text/javascript">var switchTo5x=true;</script>
<script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
<script type="text/javascript">stLight.options({publisher: "{{ config.SHARETHIS_PUBLISHER }}"});</script>
<!-- ShareThis JS -->




</head>


<body onload="prettyPrint()">

	<!-- Navbar -->
	<div class="navbar navbar-fixed-top">
		<div class="navbar-inner">
			<div class="container">

				<!-- .btn-navbar is used as the toggle for collapsed navbar content -->
				<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</a>

				<!-- Be sure to leave the brand out there if you want it shown -->
				<a class="brand" href='/'>The Mutation-Selection Blog</a>

				<!-- Everything you want hidden at 940px or less, place within here -->
				<div class="nav-collapse">
					<ul class="nav">
						<li class="dropdown">
							<a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Tags">
								<!--Tags -->
								<i class="icon-tags"></i>
								<b class="caret"></b>
							</a>
							<ul class="dropdown-menu">
								

	<li><a href='/tag/R/'>R: <small>1</small></a></li>

	<li><a href='/tag/asexual%20populations/'>asexual populations: <small>5</small></a></li>

	<li><a href='/tag/book/'>book: <small>1</small></a></li>

	<li><a href='/tag/citations/'>citations: <small>1</small></a></li>

	<li><a href='/tag/evolution%20of%20the%20mutation%20rate/'>evolution of the mutation rate: <small>4</small></a></li>

	<li><a href='/tag/evolvability/'>evolvability: <small>1</small></a></li>

	<li><a href='/tag/latex/'>latex: <small>1</small></a></li>

	<li><a href='/tag/markdown/'>markdown: <small>6</small></a></li>

	<li><a href='/tag/mathematical%20modeling/'>mathematical modeling: <small>2</small></a></li>

	<li><a href='/tag/mean%20fitness/'>mean fitness: <small>2</small></a></li>

	<li><a href='/tag/mutation%20accumulation/'>mutation accumulation: <small>1</small></a></li>

	<li><a href='/tag/mutation-selection%20balance/'>mutation-selection balance: <small>4</small></a></li>

	<li><a href='/tag/mutators/'>mutators: <small>2</small></a></li>

	<li><a href='/tag/pandoc/'>pandoc: <small>2</small></a></li>

	<li><a href='/tag/pdf/'>pdf: <small>1</small></a></li>

	<li><a href='/tag/phylogeny/'>phylogeny: <small>1</small></a></li>

	<li><a href='/tag/python/'>python: <small>2</small></a></li>

	<li><a href='/tag/recombination/'>recombination: <small>2</small></a></li>

	<li><a href='/tag/robustness/'>robustness: <small>1</small></a></li>

	<li><a href='/tag/simulation/'>simulation: <small>2</small></a></li>

	<li><a href='/tag/st2/'>st2: <small>1</small></a></li>

	<li><a href='/tag/stress-induced%20mutation/'>stress-induced mutation: <small>1</small></a></li>

	<li><a href='/tag/twitter%20bootstrap/'>twitter bootstrap: <small>1</small></a></li>


							</ul>
						</li>			
						<li class="dropdown">
							<a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Online avatars">
								<!-- Contact -->
								<i class="icon-user"></i>
								<b class="caret"></b>
							</a>
							<ul class="dropdown-menu">
								<!-- http://twitter.github.com/bootstrap/javascript.html#dropdowns -->
								
	
	<li><a href="http://www.yoavram.com/">Home <i class="icon-home"></i></a></li>
	
	
	<li><a href="mailto:yoavram@gmail.com">Email <i class="icon-envelope"></i></a></li>
	
	
	<li><a href="http://www.mendeley.com/profiles/yoav-ram">Mendeley</a></li>
	
	
	<li><a href="http://scholar.google.com/citations?user=RIFmJvYAAAAJ">Scholar Citations</a></li>
	
	
	<li><a href="http://orcid.org/0000-0002-9653-4458">ORCID</a></li>
	
	
	<li><a href="http://figshare.com/authors/Yoav_Ram/99206">FigShare</a></li>
	
	
	<li><a href="http://twitter.com/yoavram">Twitter <i class="glyphicon-twitter-t"></i></a></li>
	
	
	<li><a href="http://www.linkedin.com/in/yoavram">Linkedin <i class="glyphicon-linked-in"></i></a></li>
	
	
	<li><a href="https://github.com/yoavram">GitHub <i class="glyphicon-github"></i></a></li>
	
	
	<li><a href="https://bitbucket.org/yoavram">Bitbucket</a></li>
	
	
	<li><a href="http://code.google.com/u/yoav-ram/">Google Code</a></li>
		
				
							</ul>
						</li>
						<li>
							<a href="http://www.yoavram.com/"  title="About me">
								<i class="icon-question-sign"></i>
								&nbsp;&nbsp;
							</a>
						</li>
						<li>
							<a href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" type="application/rss+xml" title="Subscribe in a reader">
								<i class="glyphicon-rss"></i>
								&nbsp;&nbsp;				
							</a>
						</li>
						<li>
							<a href="http://feedburner.google.com/fb/a/mailverify?uri=yoavram/msb-recent" title="Subscribe by Email">
								<i class="icon-envelope"></i>
								&nbsp;&nbsp;				
							</a>
						</li>		
					</ul>
				</div>
			</div>
		</div>
	</div>
	<!-- end Navbar -->

	<!-- HEADER -->
	
<div class="row-fluid">
	<div class="span7 offset2">
	<header class="masthead">
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/citations-in-markdown-using-pandoc/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next">
		<a href='/tenaillon-et-al-2000/'>Newer &rarr;</a>
		
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#bottom">To bottom &darr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

</div>
</div>
<div class="row-fluid">
	<div class="span7 offset2">
	<h1>Mean fitness at the mutation-selection balance</h1>
	
<ul class="breadcrumb" style="background-color: #fff;padding: 0px 0;margin: 0 0 10px;">
	<li class="active"><i class="icon-time"></i> 14 October 2012 at 14:45</li>
	
	<li class="active"><span class="divider">/</span>
	<i class="icon-refresh"></i> 22 October 2012 at 07:56</li>
	
	<li><span class="divider">/</span>
	<i class="icon-star"></i> <a href='http://blog.yoavram.com/mean-fitness-at-the-mutation-selection-balance/'>Permalink</a></li>
	
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/asexual%20populations/">asexual populations</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/mathematical%20modeling/">mathematical modeling</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/mean%20fitness/">mean fitness</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/mutation-selection%20balance/">mutation-selection balance</a></li>
	
	
	
	<li><span class="divider">/</span>
	
	<!-- sharethis buttons http://sharethis.com/publishers/get-sharing-tools -->
	<span class='st_email' ></span>
	<span class='st_twitter' ></span>
	<span class='st_facebook' ></span>
	<span class='st_googleplus' ></span>
	<span class='st_reddit' ></span>
	<span class='st_linkedin' ></span>
	<span class='st_stumbleupon' ></span>
	<span class='st_delicious' ></span>
	<span class='st_citeulike' ></span>
	<span class='st_connotea' ></span>

	</li>
	</ul>

	</header>
	</div>
	<div class="span3">
		
		
<div class="well-large">
	<strong>Please cite this article as:</strong>
	<br>
	Ram, Yoav (2012) Mean fitness at the mutation-selection balance. <em>The Mutation-Selection Blog</em>. 
	<a href='http://blog.yoavram.com/mean-fitness-at-the-mutation-selection-balance/'>http://blog.yoavram.com/mean-fitness-at-the-mutation-selection-balance/</a>
</div>

		
	</div>
</div>


	<!-- BODY -->	
	<div class="row-fluid">
		<div class="span7 offset2">
			

<div class="post">
	
    <ul>
<li><a href="#overview">Overview</a></li>
<li><a href="#simple-approximation">Simple approximation</a><ul>
<li><a href="#single-locus">Single locus</a></li>
<li><a href="#multiple-locus">Multiple locus</a></li>
</ul></li>
<li><a href="#matrix-theory-the-perron-frobenius-theorem">Matrix Theory: the Perron-Frobenius theorem</a></li>
<li><a href="#summary">Summary</a></li>
<li><a href="#references">References</a></li>
</ul>
<p><a name='TOC'></a></p>
<h2 id="overview"><a href="#overview">Overview</a></h2>
<p>The first post on the <a href="http://blog.yoavram.com">Mutation-Selection Blog</a> must be about the <em>mutation-selection balance</em>, right?</p>
<p>So what is the mutation-selection balance?</p>
<p>In evolutionary biology, selection acts to remove deleterious mutations from the population, while mutation generates new deleterious mutations. When they cancel each other out, the population is at the <strong>mutation-selection balance</strong>.</p>
<p>We'll start by defining <strong>fitness</strong>. In the following treatment, the fitness of an individual is his reproductive success compared to the reproductive sucess of the most succesful individual in the population.</p>
<p>More formally, denote the fitness of individual <em>i</em> at generation <em>n</em> by <span class="math">\(\omega_{i,n}\)</span>, and define the (population) mean fitness at generation <em>n</em> by <span class="math">\(\bar{\omega}_n = E_i[\omega_{i,n}]\)</span>.</p>
<p>At the mutation-selection balance the mean fitness is at an equilibrium, that is, it doesn't change from one generation to the next: <span class="math">\[
\bar{\omega}_{n+1} = \bar{\omega}_n
\]</span> The value of this equilibrium is denoted by <span class="math">\(\bar{\omega}^*\)</span>.</p>
<p>A classical result by <span class="citation">(Kimura and Maruyama 1966)</span> involves a system in which an infinite population is evolving under <em>selection</em> and <em>mutation</em>. Mutation is unidirectional and deleterious and occurs with a rate <em>U</em>, selection is constant and uniform, the enivronment is constant, and there is no drift, sex, recombination or migration. This is a simplistic model but one that is able to capture the dynamics of some complex systems. The elegant result is that the mean fitness of a population is not dependent on selection strength and is an exponentialy decreasing function of the mutation rate <em>U</em>: <span class="math">\[
\bar{\omega}^* = e^{-U}
\]</span></p>
<h2 id="simple-approximation"><a href="#simple-approximation">Simple approximation</a></h2>
<h3 id="single-locus"><a href="#single-locus">Single locus</a></h3>
<p>In this section I roughly follow the route of <span class="citation">(Gillespie 2004)</span>. To reach this formula in a simple way (all you need is some straightforward <em>Calculus</em> and a little bit of <em>Genetics</em>), we start by looking at a single locus<sup><a href="#fn1" class="footnoteRef" id="fnref1">1</a></sup> with two alleles<sup><a href="#fn2" class="footnoteRef" id="fnref2">2</a></sup>, denoted by <em>A</em> and <em>a</em>, which have fitness of <em>1</em> and <em>1-s</em>, respectively, with <em>s</em> playing the part of the <em>selection coefficient</em> which defines the deleterious multiplicative effect a deleterious mutation has on the fitness of an individual.</p>
<p>Define <em>p</em> and <em>q</em> to be the frequencies of <em>A</em> and <em>a</em> so that <em>p+q=1</em>. Denote the probability that the wild-type allele <em>A</em> mutates to allele <em>a</em> by <span class="math">\(\mu\)</span> (without back-mutations, that is, <em>a</em> doesn't mutate back to <em>A</em>), and denote the frequency of <em>A</em> after one generation by <em>p'</em><sup><a href="#fn3" class="footnoteRef" id="fnref3">3</a></sup>: <span class="math">\[
p&#39; = \frac{p(1-\mu)}{\bar{\omega}}
\]</span> with the population mean fitness <span class="math">\(\bar{\omega} = p + q(1-s)\)</span>.</p>
<p>To find the equilibirum value of <em>p</em>, one must subtitute <span class="math">\(p&#39;=p\)</span> and <span class="math">\(q=1-p\)</span>: <span class="math">\[
p = \frac{p(1-\mu)}{p + (1-p)(1-s)} \Rightarrow \\
p(p+(1-p)(1-s)) = p(1-\mu) \Rightarrow
\]</span> we assume <span class="math">\(p\ne 0\)</span> because we are not interested in a population of mutants: <span class="math">\[
p+(1-p)(1-s) = 1-\mu \Rightarrow \\
p+1-s-p+ps = 1-\mu \Rightarrow \\
-s+ps = -\mu \Rightarrow \\ 
ps = s-\mu \Rightarrow \\
p = 1-\frac{\mu}{s} 
\]</span> So <span class="math">\(p^*=1-\frac{\mu}{s}\)</span>, that is <span class="math">\(q^*=\frac{\mu}{s}\)</span>, which makes sense - the mutant allele is produced by a rate <span class="math">\(\mu\)</span> and eliminated at a rate <span class="math">\(s\)</span>.</p>
<p>The mean fitness is therefore: <span class="math">\[
\bar{\omega}^* = p^* + (1-p^*)(1-s) = 1-\frac{\mu}{s} + \frac{\mu}{s} (1-s) =
1-\frac{\mu}{s} + \frac{\mu}{s} - s \frac{\mu}{s} \Rightarrow \\
\bar{\omega}^* = 1 - \mu 
\]</span> This, however, is the mean fitness due to a single locus, not the entire genome.</p>
<h3 id="multiple-locus"><a href="#multiple-locus">Multiple locus</a></h3>
<p>Moving on to a multiple locus model with an arbitrary number of loci<sup><a href="#fn4" class="footnoteRef" id="fnref4">4</a></sup>, we define the mutation rate at locus <em>j</em> as <span class="math">\(\mu_j\)</span>, so the mean fitness at the mutation-selection balance in locus <em>j</em> is <span class="math">\(\bar{\omega}^*_j = 1 - \mu_j\)</span>, and because fitness is multiplicative: <span class="math">\[
\bar{\omega}^* = \prod_{j}{\bar{\omega}_j^*} = \prod_{j}{1-\mu_j} \Rightarrow \\
ln{(\bar{\omega}^*)} = \sum_{j}{1-\mu_j} 
\]</span> This can be approximated, because when <span class="math">\(x\)</span> is small <span class="math">\(ln{(1-x)}\approx -x\)</span>, and <span class="math">\(\mu\)</span> is probably at most <span class="math">\(\approx 10^{-2}\)</span> or even lower: <span class="math">\[
ln{(\bar{\omega}^*)}  \approx \sum_{j}{-\mu_j} = -U
\]</span> as <em>U</em> is the mutation rate per genome per generation.</p>
<p>So This is how we can derive an approximation of the mean fitness at the mutation-selection balance: <span class="math">\[
\bar{\omega}^* = e^{-U}
\]</span></p>
<p>You can find this derivation at the begining of a <a href="http://dx.doi.org/10.6084/m9.figshare.95940">presentation</a> I gave at the <a href="http://biomathsjctau.wordpress.com">BioMath Student Journal Club at Tel-Aviv University</a>.</p>
<h2 id="matrix-theory-the-perron-frobenius-theorem"><a href="#matrix-theory-the-perron-frobenius-theorem">Matrix Theory: the Perron-Frobenius theorem</a></h2>
<p>We can produce the same result without approximations using a bit heavier mathematics, but not much heavier - this is still undergraduate <em>Linear Algebra</em>.</p>
<p>This time we start with the multiple locus model right away. Instead of following allele frequencies, though, we follow the frequency of individuals with <em>i</em> mutant alleles, or <em>i</em> deleterious mutations, which we denote by <em>p<sub>i</sub></em>.</p>
<p>We assume that the number of mutations per generation is <em>Poisson</em> distributed with the mutation rate as the parameter of the distribution, so that the probability that <em>k</em> mutations will occur in an individual with a mutaiton rate of <em>U</em> is <span class="math">\[
\varphi_U(k) = \frac{U^k e^U}{k!}
\]</span> The equation system describing the change in the freuqnecy of individuals with <em>i</em> mutant alleles is: <span class="math">\[
p&#39;_{i} = \sum_{ k=0 }^{i}{ p_{k} \frac{ \omega_{k} } {\bar{\omega}} \varphi_{U} (i-k)}, \; \; \forall i \ge 0
\]</span> where <span class="math">\(\omega_i\)</span> is the fitness of individuals with <em>i</em> mutant alleles.</p>
<p>The above equation system can be written in matrix form: <span class="math">\[
\bar{\omega}p&#39; = Mp
\]</span> where <span class="math">\(p=(p_0,p_1,p_2,...)\)</span> and <em>M</em> is: <span class="math">\[
M = \left(\begin{array}{cccc}
\omega_0 \varphi_U(0)   &amp; 0                         &amp; 0                         &amp;  ...\\
\omega_0 \varphi_U(1)   &amp; \omega_1 \varphi_U(0)     &amp; 0                         &amp;  ...\\
\omega_0 \varphi_U(2)   &amp; \omega_1 \varphi_U(1)     &amp; \omega_2 \varphi_U(0)     &amp;  ...\\ 
...                     &amp; ...                       &amp; ...                       &amp;  ...
\end{array}\right)
\]</span> This matrix is called a <a href="http://en.wikipedia.org/wiki/Stochastic_matrix">transition matrix</a> because its entries <span class="math">\(M_{i,j}\)</span> are the transition probabilities from state <em>j</em> to state <em>i</em> - the state is the number of mutant alleles.</p>
<p>Now if we set <span class="math">\(p^*=p&#39;=p\)</span> we get <span class="math">\(\bar{\omega}^*p^*=Mp^*\)</span>, so if there is <span class="math">\(\bar{\omega}^*\)</span> and <span class="math">\(p^*\)</span> such that <span class="math">\(p^*_i \ge 0 \; \sum_{i}{p^*_i}=1\)</span> that solve this equation, then <span class="math">\(p^*\)</span> is an <strong>eigenvector</strong> of <em>M</em> and <span class="math">\(\bar{\omega}^*\)</span> is an <strong>eigenvalue</strong> of M.</p>
<p>From the <a href="http://en.wikipedia.org/wiki/Perron%E2%80%93Frobenius_theorem">Perron-Frobenius theorem</a> we know that this matrix always has a unique non-negative eigenvector, so <span class="math">\(p^*\)</span> exists and it is unique. Furthermore, from the same theorem we know that <span class="math">\(\bar{\omega}^*\)</span> also exists and that it is the leading (largest positive) eigenvalue of <em>M</em> <span class="citation">(Otto and Day 2007, 709)</span>.</p>
<p>Since this is a triangular matrix, the eigenvalues are simply the elements on the diagonal<sup><a href="#fn5" class="footnoteRef" id="fnref5">5</a></sup>. Therefore, the largest eigenvalue must be the largest element on the diagonal, which is the element <span class="math">\[
\bar{\omega}^* = M_{0,0} = 
\omega_0 \varphi_U(0) = (1-s)^0 \frac{U^0 e^{-U}}{0!} = 
e^{-U}
\]</span></p>
<p>Note that if beneficial mutations or back-mutations were allowed in this model, we would not be able to derive the eigenvalue so easily, because the matrix would not be triagonal. However, it would still agree with the conditions of the Perron-Frobenius theorem and the mean fitness at the mutation-selection balance would still be the leading eigenvalue of the matrix, and would exist. I've used this fact in my paper <em>The Evolution of Stress-Induced Hypermutation in Asexual Populations</em> <span class="citation">(Ram and Hadany 2012)</span>.</p>
<h2 id="summary"><a href="#summary">Summary</a></h2>
<p>We've seen how to derive the mean fitness at the mutation-selection balance <span class="math">\(\bar{\omega} = e^{-U}\)</span> both via a first-order approximation and via a more &quot;hardcore&quot; linear algerba method.</p>
<p>The nice thing about deriving the mean fitness in the &quot;hardcore&quot; method is that we get some idea of <em>why</em> it is equal to e<sup>-U</sup>. We found that the mean fitness at the mutation-selection balance equals to the leading eigenvalue of the transition matrix, which is the relative probability that individuals from the least loaded class (the class of fittest individuals) survive to the next generation without mutating. By definition the fitness of the least loaded individuals is 1, and we assumed the mutations are <em>Poisson</em> distributed, and therefore we got e<sup>-U</sup>, but if we change these assumptions the general result will still hold:</p>
<blockquote>
<p>The mean fitness of an asexual population at the mutation-selection balance, in the absence of beneficial mutations, is the product of the fitness of the least loaded class and the probability that no mutations occur.</p>
</blockquote>
<p>What's left?</p>
<p>The next step - in the next post - will be to derive a formula for the mean fitness at each generation. This is important because we need to understand how <em>rapidly</em> the mean fitness converges towards the equilibrium value. Can we really assume that the evolutionary process has reached the mutation-selection balance in calculations we perform?</p>
<p>One article that presents an analysis on this subject is <span class="citation">(Gordo and Dionisio 2005)</span>, which shows how to derive the mean fitness of a population at genearation <em>n</em>. This is still a simple asexual population with directional selection and unidirectional deleterious mutations, but it is still very interesting, and I'll try to post a comprehensive explanation on how it can be done.</p>
<h2 id="references"><a href="#references">References</a></h2>
<p>Gillespie, John H. 2004. <em>Population Genetics: a Concise Guide</em>. 2nd ed.. Johns Hopkins University Press. <a href="http://www.google.com/books?id=KAcAfiyHpcoC" title="http://www.google.com/books?id=KAcAfiyHpcoC">http://www.google.com/books?id=KAcAfiyHpcoC</a>.</p>
<p>Gordo, Isabel, and Francisco Dionisio. 2005. &#8220;Nonequilibrium Model for Estimating Parameters of Deleterious Mutations.&#8221; <em>Physical Review E</em> 71 (3) (mar): 18&#8211;21. doi:10.1103/PhysRevE.71.031907. <a href="http://link.aps.org/doi/10.1103/PhysRevE.71.031907" title="http://link.aps.org/doi/10.1103/PhysRevE.71.031907">http://link.aps.org/doi/10.1103/PhysRevE.71.031907</a>.</p>
<p>Kimura, Motoo, and Takeo Maruyama. 1966. &#8220;The Mutational Load with Epistatic Gene Interactions in Fitness.&#8221; <em>Genetics</em> 54 (6) (dec): 1337&#8211;51. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1211299" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1211299">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1211299</a>.</p>
<p>Otto, Sarah P., and Troy Day. 2007. <em>A Biologist&#8217;s Guide to Mathematical Modeling in Ecology and Evolution</em>. Princeton University Press.</p>
<p>Ram, Yoav, and Lilach Hadany. 2012. &#8220;The Evolution of Stress-induced Hypermutation in Asexual Populations.&#8221; <em>Evolution</em> 66 (7) (jul): 2315&#8211;28. doi:10.1111/j.1558-5646.2012.01576.x. <a href="http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304" title="http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304">http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304</a>.</p>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>A site on the genome which may be occupied by different alleles. The function of a locus can be diverse - genes, transcription control sequences, etc.<a href="#fnref1">&#8617;</a></p></li>
<li id="fn2"><p>A specific sequence occupying a locus, with a specific function. One locus can potentially contain different alleles, each with a slightly different sequence and potentially drasticly different effects on function.<a href="#fnref2">&#8617;</a></p></li>
<li id="fn3"><p>A note on notation - <em>p'</em> here is <em>not</em> the derivative of <em>p</em> but rather the value of <em>p</em> at the next generation. The notation is sometimes confusing but is customary in <em>population genetics</em>. All equations presented here are <em>difference equations</em>, not <em>differential equations</em>.<a href="#fnref3">&#8617;</a></p></li>
<li id="fn4"><p>The plural of <em>locus</em>.<a href="#fnref4">&#8617;</a></p></li>
<li id="fn5"><p>To prove this, one must notice that the determinant of a triangular matrix is the trace of the matrix, and therefore the characteristic polynomial is the trace of the matrix <span class="math">\(I\lambda - M\)</span>, and the eigenvalues are the solutions of this polynomial - see <a href="http://tutorial.math.lamar.edu/Classes/LinAlg/EVals_Evects.aspx#EV_EvalEvec_Thm1">Paul's online math notes</a>.<a href="#fnref5">&#8617;</a></p></li>
</ol>
</div>
	<a href="#" id="download_bibtex_btn">Download BibTeX</a>
	<script type="text/javascript">
	$(function() {
		$('#download_bibtex_btn').click(function() {
			var content = $('#bib').html();
			var w = window.open();
			$(w.document.body).text(content);
			w.document.title = 'Mean fitness at the mutation-selection balance' + '.bib';
			w.document.contentType = 'text/x-bibtex';
		});
	})
	</script>
	<a name="bottom"></a>
	
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/citations-in-markdown-using-pandoc/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next">
		<a href='/tenaillon-et-al-2000/'>Newer &rarr;</a>
		
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#">To top &uarr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

	
</div>
<div id="bib" style="display: none">
	
@article{Kimura1966,
author = {Kimura, Motoo and Maruyama, Takeo},
file = {:D$\backslash$:/papers/Kimura, Maruyama - 1966 - The mutational load with epistatic gene interactions in fitness.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
month = dec,
number = {6},
pages = {1337--51},
pmid = {17248359},
title = {{The mutational load with epistatic gene interactions in fitness.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1211299},
volume = {54},
year = {1966}
}
@book{Otto2007,
abstract = {Thirty years ago, biologists could get by with a rudimentary grasp of mathematics and modeling. Not so today. In seeking to answer fundamental questions about how biological systems function and change over time, the modern biologist is as likely to rely on sophisticated mathematical and computer-based models as traditional fieldwork. In this book, Sarah Otto and Troy Day provide biology students with the tools necessary to both interpret models and to build their own.The book starts at an elementary level of mathematical modeling, assuming that the reader has had high school mathematics and first-year calculus. Otto and Day then gradually build in depth and complexity, from classic models in ecology and evolution to more intricate class-structured and probabilistic models. The authors provide primers with instructive exercises to introduce readers to the more advanced subjects of linear algebra and probability theory. Through examples, they describe how models have been used to understand such topics as the spread of HIV, chaos, the age structure of a country, speciation, and extinction.Ecologists and evolutionary biologists today need enough mathematical training to be able to assess the power and limits of biological models and to develop theories and models themselves. This innovative book will be an indispensable guide to the world of mathematical models for the next generation of biologists.A how-to guide for developing new mathematical models in biologyProvides step-by-step recipes for constructing and analyzing modelsInteresting biological applicationsExplores classical models in ecology and evolutionQuestions at the end of every chapterPrimers cover important mathematical topicsExercises with answersAppendixes summarize useful rulesLabs and advanced material available},
author = {Otto, Sarah P. and Day, Troy},
isbn = {9780691123448},
pages = {732},
publisher = {Princeton University Press},
title = {{A biologist's guide to mathematical modeling in ecology and evolution}},
year = {2007}
}
@article{Ram2012,
abstract = {Numerous empirical studies show that stress of various kinds induces a state of hypermutation in bacteria via multiple mechanisms, but theoretical treatment of this intriguing phenomenon is lacking. We used deterministic and stochastic models to study the evolution of stress-induced hypermutation in infinite and finite-size populations of bacteria undergoing selection, mutation, and random genetic drift in constant environments and in changing ones. Our results suggest that if beneficial mutations occur, even rarely, then stress-induced hypermutation is advantageous for bacteria at both the individual and the population levels and that it is likely to evolve in populations of bacteria in a wide range of conditions because it is favored by selection. These results imply that mutations are not, as the current view holds, uniformly distributed in populations, but rather that mutations are more common in stressed individuals and populations. Because mutation is the raw material of evolution, these results have a profound impact on broad aspects of evolution and biology.},
author = {Ram, Yoav and Hadany, Lilach},
doi = {10.1111/j.1558-5646.2012.01576.x},
file = {:D$\backslash$:/papers//Ram, Hadany - 2012 - The evolution of stress-induced hypermutation in asexual populations.pdf:pdf;:D$\backslash$:/papers//Ram, Hadany - 2012 - The evolution of stress-induced hypermutation in asexual populations.pdf:pdf},
issn = {1558-5646},
journal = {Evolution},
keywords = {Adaptation,Asexual,Bacteria,Bacteria: genetics,Bacteria: growth \& development,Biological,Computer Simulation,Environment,Genetic,Genetic Drift,Models,Mutation Rate,Physiological,Population Density,Reproduction,Stochastic Processes,Stress},
month = jul,
number = {7},
pages = {2315--28},
pmid = {22759304},
title = {{The evolution of stress-induced hypermutation in asexual populations.}},
url = {http://doi.wiley.com/10.1111/j.1558-5646.2012.01576.x http://www.ncbi.nlm.nih.gov/pubmed/22759304},
volume = {66},
year = {2012}
}
@article{Gordo2005,
abstract = {Deleterious mutations are of extreme evolutionary importance because, even though they are eliminated by natural selection, their continuous pressure creates a pool of variability in natural populations. They are of potential relevance for the existence of several features in evolution, such as sexual reproduction, and pose a risk to small asexual populations. Despite their extreme importance, the deleterious mutation rate and the effects of each mutation on fitness are poorly known quantities. Here we analyze a simple model that can be applied to simple experiments, in microorganisms, aiming at the quantification of these values.},
author = {Gordo, Isabel and Dionisio, Francisco},
doi = {10.1103/PhysRevE.71.031907},
file = {:D$\backslash$:/papers/Gordo, Dionisio - 2005 - Nonequilibrium model for estimating parameters of deleterious mutations.pdf:pdf},
issn = {1539-3755},
journal = {Physical Review E},
keywords = {muller's ratchet,mutation rate,theory},
mendeley-tags = {muller's ratchet,mutation rate,theory},
month = mar,
number = {3},
pages = {18--21},
title = {{Nonequilibrium model for estimating parameters of deleterious mutations}},
url = {http://link.aps.org/doi/10.1103/PhysRevE.71.031907},
volume = {71},
year = {2005}
}
@book{Gillespie2004,
abstract = {This concise introduction offers students and researchers an overview of the discipline that connects genetics and evolution. Addressing the theories behind population genetics and relevant empirical evidence, John Gillespie discusses genetic drift, natural selection, nonrandom mating, quantitative genetics, and the evolutionary advantage of sex. First published to wide acclaim in 1998, this brilliant primer has been updated to include new sections on molecular evolution, genetic drift, genetic load, the stationary distribution, and two-locus dynamics. This book is indispensable for students working in a laboratory setting or studying free-ranging populations.},
annote = {undefined},
author = {Gillespie, John H.},
edition = {2},
isbn = {9780801880094},
pages = {214},
publisher = {Johns Hopkins University Press},
title = {{Population genetics: a concise guide}},
url = {http://www.google.com/books?id=KAcAfiyHpcoC},
year = {2004}
}


</div>


<div id="disqus_thread"></div>
<script type="text/javascript">

	/* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
	/* see http://help.disqus.com/customer/portal/articles/472098-javascript-configuration-variables */
	var disqus_shortname = 'yoavram'; // required: replace example with your forum shortname
	var disqus_identifier = 'mean-fitness-at-the-mutation-selection-balance';
	
	/* * * DON'T EDIT BELOW THIS LINE * * */
	(function() {
		var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
		dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
		(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>





		</div>
		<div class="span3">
			<div class="sidebar">
				

		
			</div>
		</div>

	</div>

	<!-- FOOTER -->
	<div class="row-fluid">
		<div class="span8 offset2">
			<footer class="colophon"> 
				
				
				<ul class="quick-links">
					<li> 
						The content on "<span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Text" property="dct:title" rel="dct:type">The Mutation-Selection Blog</span>" by <a xmlns:cc="http://creativecommons.org/ns#" href="http://www.yoavram.com/" property="cc:attributionName" rel="cc:attributionURL">Yoav Ram</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a> license. The rest is licensed separately:
					</li>
				</ul>
				<ul class="quick-links">
					<li>Front</li>
					<li>
						<a href="http://blog.getbootstrap.com/">Twitter Bootstrap</a>
					</li>
					<li>
						<a href="http://glyphicons.com/" title="CC BY-SA 3.0">Glyphicons Free</a>
					</li>
					<li>
						<a href="http://code.google.com/p/google-code-prettify/">Google Code Prettifier</a>
					</li>
					<li>
						<a href="http://ethanschoonover.com/solarized">Solarized</a>
					</li>
					<li>
						<a href="http://mathjax.org/">MathJax</a>
					</li>
				</ul>

				<ul class="quick-links">
					<li>Back</li>
					<li><a href="http://python.org/">Python</a></li>
					<li><a href="http://flask.pocoo.org/">Flask</a></li>
					<li><a href="http://pages.github.com/">GitHub Pages</a></li>	
					<li><a href="http://daringfireball.net/projects/markdown/">Markdown</a></li>
					<li><a href="http://johnmacfarlane.net/pandoc">Pandoc</a></li>
				</ul>

				<ul class="quick-links">
					<li>Code</li>
					<li><a href="https://github.com/yoavram/yoavram.github.com">Static site</a></li>
					<li><a href="https://github.com/yoavram/yoavram.github.com/tree/source">Site generator</a></li>
					<li>Logo</li>
					<li><a href="http://www.idancohen.com/">Idan Cohen</a></li>
					<li><a href="http://www.dafont.com/sketch-block.font">Sketch Block</a></li>
					<li><a href="http://www.dafont.com/positiv-a.font">Positiv-A</a></li>
				</ul>

				<ul class="quick-links">
					<li>This page was last modified on 17 December 2013 at 15:52.</li>
				</ul>

				
				<!-- Google Analytics www.google.com/analytics/ -->
				<script type="text/javascript">

				var _gaq = _gaq || [];
				_gaq.push(['_setAccount', 'UA-3865698-10']);
				_gaq.push(['_trackPageview']);

				(function() {
					var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
					ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
					var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
				})();
				</script>
				
				<a href="/recent.atom" rel="alternate" title="Recent Articles" type="application/atom+xml"></a>
			</footer>
		</div>
	</div>

</body>
</html>