
<!DOCTYPE html>
<html lang="en">
<head>
	<title> Summary: &#34;Evolution of mutation rates in bacteria&#34; (Denamur and Matic 2006) </title>
	<meta name="description" content="Yoav Ram's blog on evolutionary theory and other scientific interests, and especially on the theoretical treatment of the evolution of the mutation rate" />
	<meta name="author" content="Yoav Ram">
	<meta name="keywords" content="evolution,mutation rate,mutation rates,Yoav Ram,yoavram,evolutionary theory,science,theoretical biology,mathematical biology,mathematics">
	<meta charset="UTF-8">
	<meta http-equiv="content-type" content="text/html; charset=UTF-8">
	<link rel="icon" type="image/ico" href="/static/favicon.ico">
	<link href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" title="Recent Articles" type="application/atom+xml">

	<!-- Le style -->
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link href='http://fonts.googleapis.com/css?family=Droid+Sans:400,700' rel='stylesheet' type='text/css'>
	<link href="/static/theme.css" rel="stylesheet">
	<link href="/static/bootstrap.min.css" rel="stylesheet">
	<link href="/static/glyphicons.css" rel="stylesheet">
	<link href="/static/bootstrap-responsive.min.css" rel="stylesheet">
		<link href="/static/jekyll.css" rel="stylesheet">

	<style type="text/css">
	[class^="icon-"] { background-image: url("/static/glyphicons-halflings.png"); } 
	[class*="icon-white"] { background-image: url("/static/glyphicons-halflings-white.png"); } 
	[class^="glyphicon-"] { background-image: url("/static/glyphicons.png"); } 
	</style>
	<link href="/static/prettify.css" rel="stylesheet">

	<!-- Le JS -->
	<script type="text/javascript" src="/static/jquery-1.7.2.min.js"></script>
	<script type="text/javascript" src="/static/bootstrap.min.js"></script>
	<script type="text/javascript">
	// http://henrywrites.wordpress.com/2012/09/03/jquery-string-format/
	String.prototype.format = String.prototype.f = function() {
	    var s = this,
	        i = arguments.length;

	    while (i--) {
	        s = s.replace(new RegExp('\\{' + i + '\\}', 'gm'), arguments[i]);
	    }
	    return s;
	};
</script>
	<!-- google code prettifier http://code.google.com/p/google-code-prettify/ -->
	<script type="text/javascript" src="/static/prettify.js"></script>

	




<!-- ShareThis JS -->
<script type="text/javascript">var switchTo5x=true;</script>
<script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
<script type="text/javascript">stLight.options({publisher: "{{ config.SHARETHIS_PUBLISHER }}"});</script>
<!-- ShareThis JS -->




</head>


<body onload="prettyPrint()">

	<!-- Navbar -->
	<div class="navbar navbar-fixed-top">
		<div class="navbar-inner">
			<div class="container">

				<!-- .btn-navbar is used as the toggle for collapsed navbar content -->
				<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</a>

				<!-- Be sure to leave the brand out there if you want it shown -->
				<a class="brand" href='/'>The Mutation-Selection Blog</a>

				<!-- Everything you want hidden at 940px or less, place within here -->
				<div class="nav-collapse">
					<ul class="nav">
						<li class="dropdown">
							<a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Tags">
								<!--Tags -->
								<i class="icon-tags"></i>
								<b class="caret"></b>
							</a>
							<ul class="dropdown-menu">
								

	<li><a href='/tag/R/'>R: <small>1</small></a></li>

	<li><a href='/tag/asexual%20populations/'>asexual populations: <small>5</small></a></li>

	<li><a href='/tag/citations/'>citations: <small>1</small></a></li>

	<li><a href='/tag/evolution%20of%20the%20mutation%20rate/'>evolution of the mutation rate: <small>4</small></a></li>

	<li><a href='/tag/evolvability/'>evolvability: <small>1</small></a></li>

	<li><a href='/tag/genetic%20drift/'>genetic drift: <small>1</small></a></li>

	<li><a href='/tag/latex/'>latex: <small>1</small></a></li>

	<li><a href='/tag/markdown/'>markdown: <small>6</small></a></li>

	<li><a href='/tag/mathematical%20modeling/'>mathematical modeling: <small>2</small></a></li>

	<li><a href='/tag/mean%20fitness/'>mean fitness: <small>2</small></a></li>

	<li><a href='/tag/muller%27s%20ratcher/'>muller&#39;s ratcher: <small>1</small></a></li>

	<li><a href='/tag/mutation%20accumulation/'>mutation accumulation: <small>1</small></a></li>

	<li><a href='/tag/mutation-selection%20balance/'>mutation-selection balance: <small>4</small></a></li>

	<li><a href='/tag/mutators/'>mutators: <small>2</small></a></li>

	<li><a href='/tag/pandoc/'>pandoc: <small>2</small></a></li>

	<li><a href='/tag/pdf/'>pdf: <small>1</small></a></li>

	<li><a href='/tag/phylogeny/'>phylogeny: <small>1</small></a></li>

	<li><a href='/tag/python/'>python: <small>2</small></a></li>

	<li><a href='/tag/recombination/'>recombination: <small>3</small></a></li>

	<li><a href='/tag/robustness/'>robustness: <small>1</small></a></li>

	<li><a href='/tag/simulation/'>simulation: <small>2</small></a></li>

	<li><a href='/tag/st2/'>st2: <small>1</small></a></li>

	<li><a href='/tag/stress-induced%20mutation/'>stress-induced mutation: <small>1</small></a></li>

	<li><a href='/tag/twitter%20bootstrap/'>twitter bootstrap: <small>1</small></a></li>


							</ul>
						</li>			
						<li class="dropdown">
							<a href="#" class="dropdown-toggle" data-toggle="dropdown" title="Online avatars">
								<!-- Contact -->
								<i class="icon-user"></i>
								<b class="caret"></b>
							</a>
							<ul class="dropdown-menu">
								<!-- http://twitter.github.com/bootstrap/javascript.html#dropdowns -->
								
	
	<li><a href="http://www.yoavram.com/">Home <i class="icon-home"></i></a></li>
	
	
	<li><a href="mailto:yoavram@gmail.com">Email <i class="icon-envelope"></i></a></li>
	
	
	<li><a href="http://www.mendeley.com/profiles/yoav-ram">Mendeley</a></li>
	
	
	<li><a href="http://scholar.google.com/citations?user=RIFmJvYAAAAJ">Scholar Citations</a></li>
	
	
	<li><a href="http://orcid.org/0000-0002-9653-4458">ORCID</a></li>
	
	
	<li><a href="http://figshare.com/authors/Yoav_Ram/99206">FigShare</a></li>
	
	
	<li><a href="http://twitter.com/yoavram">Twitter <i class="glyphicon-twitter-t"></i></a></li>
	
	
	<li><a href="http://www.linkedin.com/in/yoavram">Linkedin <i class="glyphicon-linked-in"></i></a></li>
	
	
	<li><a href="https://github.com/yoavram">GitHub <i class="glyphicon-github"></i></a></li>
	
	
	<li><a href="https://bitbucket.org/yoavram">Bitbucket</a></li>
	
	
	<li><a href="http://code.google.com/u/yoav-ram/">Google Code</a></li>
		
				
							</ul>
						</li>
						<li>
							<a href="http://www.yoavram.com/"  title="About me">
								<i class="icon-question-sign"></i>
								&nbsp;&nbsp;
							</a>
						</li>
						<li>
							<a href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" type="application/rss+xml" title="Subscribe in a reader">
								<i class="glyphicon-rss"></i>
								&nbsp;&nbsp;				
							</a>
						</li>
						<li>
							<a href="http://feedburner.google.com/fb/a/mailverify?uri=yoavram/msb-recent" title="Subscribe by Email">
								<i class="icon-envelope"></i>
								&nbsp;&nbsp;				
							</a>
						</li>		
					</ul>
				</div>
			</div>
		</div>
	</div>
	<!-- end Navbar -->

	<!-- HEADER -->
	
<div class="row-fluid">
	<div class="span7 offset2">
	<header class="masthead">
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/sniegowski-raynes-2013/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next disabled">
	   <a href='#'>Newer &rarr;</a>
	   
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#bottom">To bottom &darr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

</div>
</div>
<div class="row-fluid">
	<div class="span7 offset2">
	<h1>Summary: &#34;Evolution of mutation rates in bacteria&#34; (Denamur and Matic 2006)</h1>
	
<ul class="breadcrumb" style="background-color: #fff;padding: 0px 0;margin: 0 0 10px;">
	<li class="active"><i class="icon-time"></i> 14 July 2013 at 12:16</li>
	
	<li><span class="divider">/</span>
	<i class="icon-star"></i> <a href='http://blog.yoavram.com/denamur-and-matic-2006/'>Permalink</a></li>
	
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/evolution%20of%20the%20mutation%20rate/">evolution of the mutation rate</a></li>
	
	
	
	<li><span class="divider">/</span>
	
	<!-- sharethis buttons http://sharethis.com/publishers/get-sharing-tools -->
	<span class='st_email' ></span>
	<span class='st_twitter' ></span>
	<span class='st_facebook' ></span>
	<span class='st_googleplus' ></span>
	<span class='st_reddit' ></span>
	<span class='st_linkedin' ></span>
	<span class='st_stumbleupon' ></span>
	<span class='st_delicious' ></span>
	<span class='st_citeulike' ></span>
	<span class='st_connotea' ></span>

	</li>
	</ul>

	</header>
	</div>
	<div class="span3">
		
	</div>
</div>


	<!-- BODY -->	
	<div class="row-fluid">
		<div class="span7 offset2">
			

<div class="post">
	
    
<p>This is an &quot;executive summary&quot; of <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2006.05150.x/abstract">Denamur and Matic <span class="citation">(2006)</span></a>, which is a review of the literature on the evolution of the mutation rate in bactera.</p>
<ol style="list-style-type: decimal">
<li>Deleterious mutations are 100,000-fold more frequent than beneficial mutations in <em>E. coli</em>.</li>
<li>Mutators have been found in various species of bacteria in frequencies of 0.1-60%: 1% strong mutators, 10-30% weak mutators <span class="citation">(Matic et al. 1997; Baquero et al. 2004)</span>.</li>
<li>These frequencies are higher than expected under a <strong>mutation-selection balance</strong> (MSB) - Boe <em>et al.</em> <span class="citation">(2000)</span> estimated the mutator fraction at 0.00003.</li>
<li>Mutator alleles are mainly mutants in the <strong>mismatch repair (MMR) system</strong> (<em>mutS</em>, <em>mutL</em>) - see the article for more information on the operation of these genes and the MMR system).</li>
<li>Rate increases by these MMR mutants: <strong>100-fold increase in transitions, 1,000-fold increase in frameshifts and 10-1,000-fold increase in chromosomal rearrangements</strong>.</li>
<li>MMR mutatns arise by different mutation types. The rate of non-mutator to mutator was estimated by Boe <em>et al.</em> <span class="citation">(2000)</span> to be 0.000005 per generation.</li>
<li>What can cause the frequency of MMR mutants to be higher than that expected under a MSB?
<ul>
<li><strong>Higher replication rates</strong> due to the absence of the metabolic load imposed by DNA repair enzymes<sup><a href="#fn1" class="footnoteRef" id="fnref1">1</a></sup></li>
<li><strong>Higher adaptation rate</strong> due to the faster generation of beneficial mutations<sup><a href="#fn2" class="footnoteRef" id="fnref2">2</a></sup></li>
</ul></li>
<li><strong>The first explanation was rejected</strong>: Several studies found that <strong>mutators are advanageous only when</strong> the ratio of mutator to non-mutator is such that <strong>beneficial mutations are more likely to be generated by mutators</strong> than by non-mutators<sup><a href="#fn3" class="footnoteRef" id="fnref3">3</a></sup>.</li>
<li>The second explanation is stonger in bacteria than in other species because <strong>recombination</strong> rates are low and therefore mutators are not separated from the beneficial mutations they generate.</li>
<li>Although mutators can reach <strong>high frequencies in adaptive evolution</strong>, they accumulate deleterious mutations and <strong>decline in frequency in a constant environment</strong>.</li>
<li><strong>Migration</strong> to new environments can change a beneficial mutation to neutral or deleterious.</li>
<li>Mutators are more vulnerable to <strong>Muller's ratchet</strong> due to faster accumulation of deleterious mutations <span class="citation">(Funchain et al. 2000)</span>.</li>
<li>Beneficial mutations can move to non-mutator background by horizontal gene transfer or back/compensatory mutation at the mutator locus.</li>
<li><strong>Local mutators</strong> - DNA sequences that induce high mutation rates in their neighborhoods - were found in virulent loci.</li>
<li>There are over 20 loci associated with mutator phenotypes, probably with different direct effects on mutation rate and fitness and with different <strong>pleiotropic effects</strong> - for example, <em>mutT</em> increases the mutation and adaptation rate but also increases transcriptional error rate <span class="citation">(Taddei, Hayakawa, et al. 1997)</span>.</li>
<li>MMR mutants are special because they also <strong>increase recombination rates</strong> which can help in adaptation <span class="citation">(Funchain et al. 2001)</span>. High recombination rates can also help to restore non-mutator alleles after adaptation is complete<sup><a href="#fn4" class="footnoteRef" id="fnref4">4</a></sup>.</li>
<li>Most mutators were found in <strong>pathogenic bacteria</strong>.</li>
<li>Some mutators were correlated in natural populations with <strong>antibiotic resistance</strong>, however not strong mutators: the latter are probably counter-selected after adaptation and are therefore not found in natural population <span class="citation">(Taddei, Radman, et al. 1997)</span>.</li>
</ol>
<h1 id="references">References</h1>
<p>Baquero, Mar&#237;a-Rosario, Annika I. Nilsson, Mar&#237;a del Carmen Turrientes, Dorthe Sandvang, Juan Carlos Gal&#225;n, Jose Lu&#237;s Mart&#237;nez, Niels Frimodt-M&#248; ller, Fernando Baquero, and Dan I. Andersson. 2004. &#8220;Polymorphic Mutation Frequencies in Escherichia Coli: Emergence of Weak Mutators in Clinical Isolates.&#8221; <em>Journal of Bacteriology</em> 186 (16) (aug): 5538&#8211;42. doi:10.1128/JB.186.16.5538-5542.2004. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=490890" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=490890">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=490890</a>.</p>
<p>Boe, L., M. Danielsen, S. Knudsen, J. B. Petersen, J. Maymann, and P. R. Jensen. 2000. &#8220;The Frequency of Mutators in Populations of Escherichia Coli.&#8221; <em>Mutation Research</em> 448 (1) (mar): 47&#8211;55. doi:10.1016/S0027-5107(99)00239-0. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10751622" title="http://www.ncbi.nlm.nih.gov/pubmed/10751622">http://www.ncbi.nlm.nih.gov/pubmed/10751622</a>.</p>
<p>Dawson, Kevin J. 1998. &#8220;Evolutionarily Stable Mutation Rates.&#8221; <em>Journal of Theoretical Biology</em> 194 (1) (sep): 143&#8211;57. doi:10.1006/jtbi.1998.0752. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9778430" title="http://www.ncbi.nlm.nih.gov/pubmed/9778430">http://www.ncbi.nlm.nih.gov/pubmed/9778430</a>.</p>
<p>Denamur, Erick, and Ivan Matic. 2006. &#8220;Evolution of Mutation Rates in Bacteria.&#8221; <em>Molecular Microbiology</em> 60 (4) (may): 820&#8211;7. doi:10.1111/j.1365-2958.2006.05150.x. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16677295" title="http://www.ncbi.nlm.nih.gov/pubmed/16677295">http://www.ncbi.nlm.nih.gov/pubmed/16677295</a>.</p>
<p>Funchain, P., A. Yeung, J. Stewart, W. M. Clendenin, and J. H. Miller. 2001. &#8220;Amplification of Mutator Cells in a Population as a Result of Horizontal Transfer.&#8221; <em>Journal of Bacteriology</em> 183 (12) (jun): 3737&#8211;41. doi:10.1128/JB.183.12.3737-3741.2001. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=95251" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=95251">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=95251</a>.</p>
<p>Funchain, Pauline, Annie Yeung, Jean Lee Stewart, Rose Lin, Malgorzata M. Slupska, and Jeffrey H. Miller. 2000. &#8220;The Consequences of Growth of a Mutator Strain of Escherichia Coli as Measured by Loss of Function Among Multiple Gene Targets and Loss of Fitness.&#8221; <em>Genetics</em> 154 (3) (mar): 959&#8211;70. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1461004" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1461004">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1461004</a>.</p>
<p>Leigh, Egbert Giles. 1973. &#8220;The Evolution of Mutation Rates.&#8221; <em>Genetics</em> 73 (apr): Suppl73:1&#8211;18. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4711556" title="http://www.ncbi.nlm.nih.gov/pubmed/4711556">http://www.ncbi.nlm.nih.gov/pubmed/4711556</a>.</p>
<p>Matic, Ivan, Fran&#231;ois Taddei, Bertrand Picard, Catherine Doit, Edouard Bingen, Erick Denamur, and Jacques Elion. 1997. &#8220;Highly Variable Mutation Rates in Commensal and Pathogenic Escherichia Coli.&#8221; <em>Science</em> 277 (5333) (sep): 1833&#8211;1834. doi:10.1126/science.277.5333.1833. <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.277.5333.1833" title="http://www.sciencemag.org/cgi/doi/10.1126/science.277.5333.1833">http://www.sciencemag.org/cgi/doi/10.1126/science.277.5333.1833</a>.</p>
<p>Taddei, Fran&#231;ois, H. Hayakawa, M. Bouton, A. Cirinesi, I. Matic, M. Sekiguchi, and M. Radman. 1997. &#8220;Counteraction by MutT Protein of Transcriptional Errors Caused by Oxidative Damage.&#8221; <em>Science</em> 278 (5335) (oct): 128&#8211;30. doi:10.1126/science.278.5335.128. <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.278.5335.128 http://www.ncbi.nlm.nih.gov/pubmed/9311918" title="http://www.sciencemag.org/cgi/doi/10.1126/science.278.5335.128 http://www.ncbi.nlm.nih.gov/pubmed/9311918">http://www.sciencemag.org/cgi/doi/10.1126/science.278.5335.128 http://www.ncbi.nlm.nih.gov/pubmed/9311918</a>.</p>
<p>Taddei, Fran&#231;ois, Miroslav Radman, John Maynard Smith, Bruno Toupance, Pierre-Henri Gouyon, and Bernard Godelle. 1997. &#8220;Role of Mutator Alleles in Adaptive Evolution.&#8221; <em>Nature</em> 387 (6634) (jun): 700&#8211;2. doi:10.1038/42696. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9192893" title="http://www.ncbi.nlm.nih.gov/pubmed/9192893">http://www.ncbi.nlm.nih.gov/pubmed/9192893</a>.</p>
<p>Torres-Barcel&#243;, Clara, Gabriel Cabot, Antonio Oliver, Angus Buckling, and R. Craig Maclean. 2013. &#8220;A Trade-off Between Oxidative Stress Resistance and DNA Repair Plays a Role in the Evolution of Elevated Mutation Rates in Bacteria.&#8221; <em>Proceedings. Biological Sciences / The Royal Society</em> 280 (1757) (jan): 20130007. doi:10.1098/rspb.2013.0007. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23446530 http://datadryad.org/resource/doi:10.5061/dryad.jr4dc" title="http://www.ncbi.nlm.nih.gov/pubmed/23446530 http://datadryad.org/resource/doi:10.5061/dryad.jr4dc">http://www.ncbi.nlm.nih.gov/pubmed/23446530 http://datadryad.org/resource/doi:10.5061/dryad.jr4dc</a>.</p>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Cost of DNA replication fidelity <span class="citation">(Dawson 1998)</span><a href="#fnref1">&#8617;</a></p></li>
<li id="fn2"><p>Second-order selection (for example, <span class="citation">(Leigh 1973)</span>).<a href="#fnref2">&#8617;</a></p></li>
<li id="fn3"><p>But see <span class="citation">(Torres-Barcel&#243; et al. 2013)</span> which showed that <em>P. aeruginosa</em> mutators are oxidative-stress resistant.<a href="#fnref3">&#8617;</a></p></li>
<li id="fn4"><p>Rise and fall of the mutator allele <span class="citation">(Taddei, Radman, et al. 1997)</span><a href="#fnref4">&#8617;</a></p></li>
</ol>
</div>
	<a href="#" id="download_bibtex_btn">Download BibTeX</a>
	<script type="text/javascript">
	$(function() {
		$('#download_bibtex_btn').click(function() {
			var content = $('#bib').html();
			var w = window.open();
			$(w.document.body).text(content);
			w.document.title = 'Summary: &#34;Evolution of mutation rates in bacteria&#34; (Denamur and Matic 2006)' + '.bib';
			w.document.contentType = 'text/x-bibtex';
		});
	})
	</script>
	<a name="bottom"></a>
	
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/sniegowski-raynes-2013/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next disabled">
	   <a href='#'>Newer &rarr;</a>
	   
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#">To top &uarr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

	
</div>
<div id="bib" style="display: none">
	
@article{Denamur2006,
abstract = {Evolutionary success of bacteria relies on the constant fine-tuning of their mutation rates, which optimizes their adaptability to constantly changing environmental conditions. When adaptation is limited by the mutation supply rate, under some conditions, natural selection favours increased mutation rates by acting on allelic variation of the genetic systems that control fidelity of DNA replication and repair. Mutator alleles are carried to high frequency through hitchhiking with the adaptive mutations they generate. However, when fitness gain no longer counterbalances the fitness loss due to continuous generation of deleterious mutations, natural selection favours reduction of mutation rates. Selection and counter-selection of high mutation rates depends on many factors: the number of mutations required for adaptation, the strength of mutator alleles, bacterial population size, competition with other strains, migration, and spatial and temporal environmental heterogeneity. Such modulations of mutation rates may also play a role in the evolution of antibiotic resistance.},
author = {Denamur, Erick and Matic, Ivan},
doi = {10.1111/j.1365-2958.2006.05150.x},
file = {:D$\backslash$:/papers/Denamur, Matic - 2006 - Evolution of mutation rates in bacteria.pdf:pdf},
issn = {0950-382X},
journal = {Molecular microbiology},
keywords = {Bacteria,Bacteria: drug effects,Bacteria: genetics,Bacteria: pathogenicity,Bacterial,Bacterial: genetics,Base Pair Mismatch,Base Pair Mismatch: genetics,Base Sequence,DNA Repair,DNA Repair: genetics,Drug Resistance,Evolution,Molecular,Molecular Sequence Data,Mutagenesis,Mutation,bacteria,mutation rate,review},
mendeley-tags = {bacteria,mutation rate,review},
month = may,
number = {4},
pages = {820--7},
pmid = {16677295},
title = {{Evolution of mutation rates in bacteria.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/16677295},
volume = {60},
year = {2006}
}
@article{Baquero2004,
abstract = {Polymorphisms in the rifampin resistance mutation frequency (f) were studied in 696 Escherichia coli strains from Spain, Sweden, and Denmark. Of the 696 strains, 23\% were weakly hypermutable (4 x 10(-8) < or = f < 4 x 10(-7)), and 0.7\% were strongly hypermutable (f > or = 4 x 10(-7)). Weak mutators were apparently more frequent in southern Europe and in blood isolates (38\%) than in urinary tract isolates (25\%) and feces of healthy volunteers (11\%).},
author = {Baquero, Mar\'{\i}a-Rosario and Nilsson, Annika I and Turrientes, Mar\'{\i}a del Carmen and Sandvang, Dorthe and Gal\'{a}n, Juan Carlos and Mart\'{\i}nez, Jose Lu\'{\i}s and Frimodt-M\o ller, Niels and Baquero, Fernando and Andersson, Dan I},
doi = {10.1128/JB.186.16.5538-5542.2004},
issn = {0021-9193},
journal = {Journal of bacteriology},
keywords = {Anti-Bacterial Agents,Anti-Bacterial Agents: pharmacology,Bacterial,Bacterial: genetics,Blood,Blood: microbiology,Denmark,Drug Resistance,Escherichia coli,Escherichia coli Infections,Escherichia coli Infections: microbiology,Escherichia coli: drug effects,Escherichia coli: genetics,Escherichia coli: isolation \& purification,Feces,Feces: microbiology,Genes,Genetic,Humans,Mutation,Polymorphism,Rifampin,Rifampin: pharmacology,Spain,Sweden,Urinary Tract Infections,Urinary Tract Infections: microbiology,epidemiology,mutator},
mendeley-tags = {epidemiology,mutator},
month = aug,
number = {16},
pages = {5538--42},
pmid = {15292159},
title = {{Polymorphic mutation frequencies in Escherichia coli: emergence of weak mutators in clinical isolates.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=490890},
volume = {186},
year = {2004}
}
@article{Dawson1998,
abstract = {I investigate the hypothesis that mutation rates in natural populations are determined by a balance between: (1) selection against deleterious mutations favouring lower mutation rates, and (2) selection opposing further reduction of the mutation rate, resulting from the costs incurred by more stringent proof-reading and repair (for example, a reduction in the rate of DNA replication). The influence of advantageous mutations is assumed to be negligible. In a previous paper, I analysed the dynamics of a modifier of the mutation rate in a large sexual population, where (infinitesimally rare) deleterious alleles segregate at an infinite number of unlinked loci with symmetric multiplicative fitness effects. A simple condition was obtained for a modifier allele to increase in frequency. Remarkably, this condition does not depend on the allele frequencies at the modifier locus. Here, I show that (as a consequence), given any set of possible values of the mutation rate (any set of possible modifier alleles), there always exists a single globally stable value of the mutation rate. This is an unusually strong form of "evolutionary stability" for a sexual population. Less surprisingly the optimum mutation rate in an asexual population has similar stability properties. Furthermore, in the case of an asexual population, it is not necessary to make any special assumptions about the selection acting against deleterious mutations, except that a deterministic mutation-selection equilibrium exists. I present a simple method for identifying the evolutionarily stable value of the mutation rate, given the function alpha(U) relating the value of the mutation rate to the fitness cost of maintaining this rate. I also argue that if there is a highly conserved relationship between the rate of replication per base, and the rate of mutation per base, and if this relationship has the form of a power law, then the remarkable uniformity of the per genome mutation rate in DNA based microbes can be explained.},
annote = {no beneficial mutations
MSB
the difference between k-m model and kond model
stable mu rate is lower for asex (like sloan \& panjeti) because indirect selection is weaker. very different results than if ben muts are allowed (leigh 70) in which case asex will have higher mut rates.selection at haploid stage - no hetero effect
        
        
discussion on the different terms of the CoF
        
        
      },
author = {Dawson, Kevin J.},
doi = {10.1006/jtbi.1998.0752},
file = {:D$\backslash$:/papers//Dawson - 1998 - Evolutionarily stable mutation rates.pdf:pdf},
issn = {0022-5193},
journal = {Journal of theoretical biology},
keywords = {Animals,Asexual,Asexual: genetics,Evolution,Genetic,Molecular,Mutation,Reproduction,Reproduction: genetics,Selection,cost of fidelity,mutation rate,theory},
mendeley-tags = {cost of fidelity,mutation rate,theory},
month = sep,
number = {1},
pages = {143--57},
pmid = {9778430},
title = {{Evolutionarily stable mutation rates.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/9778430},
volume = {194},
year = {1998}
}
@article{Matic1997,
author = {Matic, Ivan and Taddei, Fran\c{c}ois and Picard, Bertrand and Doit, Catherine and Bingen, Edouard and Denamur, Erick and Elion, Jacques},
doi = {10.1126/science.277.5333.1833},
file = {:D$\backslash$:/papers/Matic et al. - 1997 - Highly Variable Mutation Rates in Commensal and Pathogenic Escherichia coli.pdf:pdf},
issn = {00368075},
journal = {Science},
keywords = {mutation rate},
mendeley-tags = {mutation rate},
month = sep,
number = {5333},
pages = {1833--1834},
title = {{Highly Variable Mutation Rates in Commensal and Pathogenic Escherichia coli}},
url = {http://www.sciencemag.org/cgi/doi/10.1126/science.277.5333.1833},
volume = {277},
year = {1997}
}
@article{Boe2000,
abstract = {Owing to occasional spontaneous mutations in genes encoding DNA repair, any population of a reasonable size is expected to harbor a sub-population of genetic mutators. Using a genetically modified strain of Escherichia coli K-12, we have estimated the frequency of mutators to be about 3x10(-5). By and large, this corresponds to a mutation rate from non-mutators to mutators of 5x10(-6) per bacterium per generation. Using a mutS\&Colon;Tn10 derivative as representative for mutators, we estimated the increase in mutation rates in mutators to be 19- to 82-fold, depending on the test-mutation under consideration. The load associated with this increase in mutation rate resulted in a growth inhibition of 1\%. From these data, we estimated that the rate of detrimental mutations in the non-mutators to be 2x10(-4)-8x10(-4). The situations where adaptive mutations may result in an increase in the frequency of mutators are discussed.},
annote = {mutator freq 3e-5
rate from um to mutator 5e-6 per bacterium per generation
modifier: 19-82 fold
selective disadvantage of mutator: \%1},
author = {Boe, L and Danielsen, M and Knudsen, S and Petersen, J B and Maymann, J and Jensen, P R},
doi = {10.1016/S0027-5107(99)00239-0},
file = {:D$\backslash$:/papers/Boe et al. - 2000 - The frequency of mutators in populations of Escherichia coli.pdf:pdf},
issn = {0027-5107},
journal = {Mutation research},
keywords = {Adenosine Triphosphatases,Bacterial Proteins,Bacterial Proteins: genetics,Cell Division,Cell Division: genetics,Culture Media,DNA Repair,DNA Repair: genetics,DNA-Binding Proteins,Escherichia coli,Escherichia coli Proteins,Escherichia coli: genetics,Escherichia coli: metabolism,Gene Frequency,Genetic,Genetics,Lac Repressors,Lactose,Lactose: metabolism,MutS DNA Mismatch-Binding Protein,Mutation,Population,Repressor Proteins,Repressor Proteins: genetics,Selection,bacteria,mutation rate,mutator},
mendeley-tags = {bacteria,mutation rate,mutator},
month = mar,
number = {1},
pages = {47--55},
pmid = {10751622},
publisher = {Elsevier},
title = {{The frequency of mutators in populations of Escherichia coli.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/10751622},
volume = {448},
year = {2000}
}
@article{Leigh1973,
author = {Leigh, Egbert Giles Jr.},
file = {:D$\backslash$:/papers/Leigh - 1973 - The evolution of mutation rates.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
keywords = {Adaptation,Alleles,Biological,Biological Evolution,Environment,Genes,Genetic,Lethal,Mutation,Selection,mutation rate,theory},
mendeley-tags = {mutation rate,theory},
month = apr,
pages = {Suppl 73:1--18},
pmid = {4711556},
title = {{The evolution of mutation rates.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/4711556},
volume = {73},
year = {1973}
}
@article{Taddei1997,
abstract = {Because most newly arising mutations are neutral or deleterious, it has been argued that the mutation rate has evolved to be as low as possible, limited only by the cost of error-avoidance and error-correction mechanisms. But up to one per cent of natural bacterial isolates are 'mutator' clones that have high mutation rates. We consider here whether high mutation rates might play an important role in adaptive evolution. Models of large, asexual, clonal populations adapting to a new environment show that strong mutator genes (such as those that increase mutation rates by 1,000-fold) can accelerate adaptation, even if the mutator gene remains at a very low frequency (for example, 10[-5]). Less potent mutators (10 to 100-fold increase) can become fixed in a fraction of finite populations. The parameters of the model have been set to values typical for Escherichia coli cultures, which behave in a manner similar to the model in long-term adaptation experiments.},
annote = {-transient mutators:  by mutation and reverse mutation at modifier gene. rate: \~{} 10-5-10-7
-if transient effect is removed, that is, no reversion allowed,  than mutators do not speed adaptation
-"Furthermore, these results show that during the course of evolution, where phases of adaptation and stasis alternate, there is no pure strategy of mutation rate. Rather, selection seems to result in an alternation of high and low mutation rate through forward and reverse mutations at the mutator locus. Alternatively, transient mutators
ability by increasing their mutation rate: for example, inducible mechanisms (see ref. 30 for a review) might be particularly useful in response to stress."
      },
author = {Taddei, Fran\c{c}ois and Radman, Miroslav and {Maynard Smith}, John and Toupance, Bruno and Gouyon, Pierre-Henri and Godelle, Bernard},
doi = {10.1038/42696},
file = {:D$\backslash$:/papers/Taddei et al. - 1997 - Role of mutator alleles in adaptive evolution.pdf:pdf},
issn = {0028-0836},
journal = {Nature},
keywords = {Adaptation,Alleles,Animals,Biological,Directed Molecular Evolution,Escherichia coli,Evolution,Genetic,Mammals,Models,Molecular,Mutation,Physiological,Selection,mutator},
mendeley-tags = {mutator},
month = jun,
number = {6634},
pages = {700--2},
pmid = {9192893},
title = {{Role of mutator alleles in adaptive evolution.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/9192893},
volume = {387},
year = {1997}
}
@article{Funchain2000,
abstract = {We have examined the composition of members of mutator populations of Escherichia coli by employing an extensive set of phenotypic screens that allow us to monitor the function of >700 genes, constituting approximately 15\% of the genome. We looked at mismatch repair deficient cells after repeated cycles of single colony isolation on rich medium to generate lineages that are forced through severe bottlenecks, and compared the results to those for wild-type strains. The mutator lineages continued to accumulate mutations rapidly with each increasing cycle of colony isolation. By the end of the 40th cycle, after approximately 1000 generations, most of the lineages had reduced colony size, 4\% had died out, 55\% had auxotrophic requirements (increasing to 80\% after 60 cycles), and 70\% had defects in at least one sugar or catabolic pathway. In addition, 33\% had a defect in cell motility, and 26\% were either temperature-sensitive or cold-sensitive lethals. On the other hand, only 3\% of the wild-type lineages had detectable mutations of any type after 40 cycles. By the 60th cycle, the typical mutator cell carried 4-5 inactive genes among the 15\% of the genome being monitored, indicating that the average cell carried at least 24-30 inactivated genes distributed throughout the genome. Remarkably, 30\% of the lineages had lost the ability to utilize xylose as a carbon source. DNA sequencing revealed that most of the Xyl(-) mutants had a frameshift in a run of eight G's (GGGGGGGG) in the xylB gene, either adding or deleting one -G-. Further analysis indicated that rendering E. coli deficient in mismatch repair unmasks hypermutable sites in certain genes or intergenic regions. Growth curves and competition tests on lineages that passed through 90 cycles of single colony isolation showed that all lineages suffered reduced fitness. We discuss these results in terms of the value of mutators in cellular evolution.},
annote = {-the typical mutator cell carried ... at least 24–30 inactivated genes distributed throughout the genome. 
-all lineages suffered reduced fitness
-approximated by phenotypic screen on >700 genes
        
      },
author = {Funchain, Pauline and Yeung, Annie and Stewart, Jean Lee and Lin, Rose and Slupska, Malgorzata M. and Miller, Jeffrey H.},
file = {:D$\backslash$:/papers/Funchain et al. - 2000 - The consequences of growth of a mutator strain of Escherichia coli as measured by loss of function among multiple gene targets and loss of fitness.pdf:pdf},
issn = {0016-6731},
journal = {Genetics},
keywords = {Adaptation,Bacterial,Biological,Escherichia coli,Escherichia coli: genetics,Escherichia coli: growth \& development,Genes,Mutation,bacteria,escherichia coli,fitness,in vivo,mutation accumulation,mutator},
mendeley-tags = {bacteria,escherichia coli,fitness,in vivo,mutation accumulation,mutator},
month = mar,
number = {3},
pages = {959--70},
pmid = {10757746},
title = {{The consequences of growth of a mutator strain of Escherichia coli as measured by loss of function among multiple gene targets and loss of fitness.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1461004},
volume = {154},
year = {2000}
}
@article{Funchain2001,
abstract = {Mutator cells that lack the mismatch repair system (MMR(-)) occur at rates of 10(-5) or less in laboratory populations started from wild-type cells. We show that after selection for recombinants in an interspecies mating between Salmonella enterica serovar Typhimurium and Escherichia coli, the percentage of MMR(-) cells rises to several percent of the recombinant population, and after a second successive mating and selection, greater than 95\% of the recombinants are MMR(-). Coupling a single cross and selection with either mutagenesis or selection for spontaneous mutants also results in a dramatic increase in MMR(-) cells. We discuss how horizontal transfer can result in mutator strains during adaptive evolution.},
author = {Funchain, P and Yeung, A and Stewart, J and Clendenin, W M and Miller, J H},
doi = {10.1128/JB.183.12.3737-3741.2001},
issn = {0021-9193},
journal = {Journal of bacteriology},
keywords = {Base Pair Mismatch,Conjugation,Crosses,Escherichia coli,Escherichia coli: cytology,Escherichia coli: genetics,Gene Transfer,Genetic,Horizontal,Phenotype,Salmonella enterica,Salmonella enterica: cytology,Salmonella enterica: genetics,mutator},
mendeley-tags = {mutator},
month = jun,
number = {12},
pages = {3737--41},
pmid = {11371538},
title = {{Amplification of mutator cells in a population as a result of horizontal transfer.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=95251},
volume = {183},
year = {2001}
}


</div>


<div id="disqus_thread"></div>
<script type="text/javascript">

	/* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
	/* see http://help.disqus.com/customer/portal/articles/472098-javascript-configuration-variables */
	var disqus_shortname = 'yoavram'; // required: replace example with your forum shortname
	var disqus_identifier = 'denamur-and-matic-2006';
	
	/* * * DON'T EDIT BELOW THIS LINE * * */
	(function() {
		var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
		dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
		(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>





		</div>
		<div class="span3">
			<div class="sidebar">
				

		
			</div>
		</div>

	</div>

	<!-- FOOTER -->
	<div class="row-fluid">
		<div class="span8 offset2">
			<footer class="colophon"> 
				
				
				<ul class="quick-links">
					<li> 
						The content on "<span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Text" property="dct:title" rel="dct:type">The Mutation-Selection Blog</span>" by <a xmlns:cc="http://creativecommons.org/ns#" href="http://www.yoavram.com/" property="cc:attributionName" rel="cc:attributionURL">Yoav Ram</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a> license. The rest is licensed separately:
					</li>
				</ul>
				<ul class="quick-links">
					<li>Front</li>
					<li>
						<a href="http://blog.getbootstrap.com/">Twitter Bootstrap</a>
					</li>
					<li>
						<a href="http://glyphicons.com/" title="CC BY-SA 3.0">Glyphicons Free</a>
					</li>
					<li>
						<a href="http://code.google.com/p/google-code-prettify/">Google Code Prettifier</a>
					</li>
					<li>
						<a href="http://ethanschoonover.com/solarized">Solarized</a>
					</li>
					<li>
						<a href="http://mathjax.org/">MathJax</a>
					</li>
				</ul>

				<ul class="quick-links">
					<li>Back</li>
					<li><a href="http://python.org/">Python</a></li>
					<li><a href="http://flask.pocoo.org/">Flask</a></li>
					<li><a href="http://pages.github.com/">GitHub Pages</a></li>	
					<li><a href="http://daringfireball.net/projects/markdown/">Markdown</a></li>
					<li><a href="http://johnmacfarlane.net/pandoc">Pandoc</a></li>
				</ul>

				<ul class="quick-links">
					<li>Code</li>
					<li><a href="https://github.com/yoavram/yoavram.github.com">Static site</a></li>
					<li><a href="https://github.com/yoavram/yoavram.github.com/tree/source">Site generator</a></li>
					<li>Logo</li>
					<li><a href="http://www.idancohen.com/">Idan Cohen</a></li>
					<li><a href="http://www.dafont.com/sketch-block.font">Sketch Block</a></li>
					<li><a href="http://www.dafont.com/positiv-a.font">Positiv-A</a></li>
				</ul>

				<ul class="quick-links">
					<li>This page was last modified on 15 July 2013 at 11:07.</li>
				</ul>

				
				<!-- Google Analytics www.google.com/analytics/ -->
				<script type="text/javascript">

				var _gaq = _gaq || [];
				_gaq.push(['_setAccount', 'UA-3865698-10']);
				_gaq.push(['_trackPageview']);

				(function() {
					var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
					ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
					var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
				})();
				</script>
				
				<a href="/recent.atom" rel="alternate" title="Recent Articles" type="application/atom+xml"></a>
			</footer>
		</div>
	</div>

</body>
</html>