
<!DOCTYPE html>
<html lang="en">
<head>
<title> Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000) </title>
<meta name="description" content="Yoav Ram's blog on evolutionary theory and other scientific interests, and especially on the theoretical treatment of the evolution of the mutation rate" />
<meta name="author" content="Yoav Ram">
<meta name="keywords" content="evolution,mutation rate,mutation rates,Yoav Ram,yoavram,evolutionary theory,science,theoretical biology,mathematical biology,mathematics">
<meta charset="UTF-8">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="icon" type="image/ico" href="/static/favicon.ico">
<link href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" title="Recent Articles" type="application/atom+xml">

<!-- Le style -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link href="/static/theme.css" rel="stylesheet">
<link href="/static/jekyll.css" rel="stylesheet">
<link href="/static/bootstrap.min.css" rel="stylesheet">
<link href="/static/glyphicons.css" rel="stylesheet">
<link href="/static/bootstrap-responsive.min.css" rel="stylesheet">
<style type="text/css">
[class^="icon-"] { background-image: url("/static/glyphicons-halflings.png"); } 
[class*="icon-white"] { background-image: url("/static/glyphicons-halflings-white.png"); } 
[class^="glyphicon-"] { background-image: url("/static/glyphicons.png"); } 
</style>
<link href="/static/prettify.css" rel="stylesheet">

<!-- Le JS -->
<script type="text/javascript" src="/static/jquery-1.7.2.min.js"></script>
<script type="text/javascript" src="/static/bootstrap.min.js"></script>

<!-- google code prettifier http://code.google.com/p/google-code-prettify/ -->
<script type="text/javascript" src="/static/prettify.js"></script>





<!-- MathJax -->
<script type="text/x-mathjax-config">
<!-- Online Debugger: http://www.onemathematicalcat.org/MathJaxDocumentation/TeXSyntax.htm -->
<!-- Doc: http://docs.mathjax.org/en/latest/output.html -->`
  MathJax.Hub.Config({            
			"HTML-CSS": { availableFonts: ["STIX", "TeX"], 
						  linebreaks: { automatic: true }
			}
  });
</script>
<script type="text/javascript"
  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML">
</script>




<!-- ShareThis JS -->
<script type="text/javascript">var switchTo5x=true;</script>
<script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
<script type="text/javascript">stLight.options({publisher: "{{ config.SHARETHIS_PUBLISHER }}"});</script>
<!-- ShareThis JS -->




</head>


<body onload="prettyPrint()">

<!-- Navbar -->
<div class="navbar navbar-fixed-top">
  <div class="navbar-inner">
    <div class="container">
 
      <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
      <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </a>
 
      <!-- Be sure to leave the brand out there if you want it shown -->
      <a class="brand" href='/'>The Mutation-Selection Blog</a>
 
      <!-- Everything you want hidden at 940px or less, place within here -->
      <div class="nav-collapse">
        <ul class="nav">
			<li class="dropdown">
				<a href="#" class="dropdown-toggle" data-toggle="dropdown"
				title="Tags">
					<!--Tags -->
					<i class="icon-tags"></i>
					<b class="caret"></b>
				</a>
				<ul class="dropdown-menu">
				

	<li><a href='/tag/R/'>R: <small>1</small></a></li>

	<li><a href='/tag/asexual%20populations/'>asexual populations: <small>3</small></a></li>

	<li><a href='/tag/citations/'>citations: <small>1</small></a></li>

	<li><a href='/tag/evolution%20of%20the%20mutation%20rate/'>evolution of the mutation rate: <small>1</small></a></li>

	<li><a href='/tag/latex/'>latex: <small>1</small></a></li>

	<li><a href='/tag/markdown/'>markdown: <small>6</small></a></li>

	<li><a href='/tag/mathematical%20modeling/'>mathematical modeling: <small>2</small></a></li>

	<li><a href='/tag/mean%20fitness/'>mean fitness: <small>2</small></a></li>

	<li><a href='/tag/mutation-selection%20balance/'>mutation-selection balance: <small>2</small></a></li>

	<li><a href='/tag/mutators/'>mutators: <small>1</small></a></li>

	<li><a href='/tag/pandoc/'>pandoc: <small>1</small></a></li>

	<li><a href='/tag/pdf/'>pdf: <small>1</small></a></li>

	<li><a href='/tag/phylogeny/'>phylogeny: <small>1</small></a></li>

	<li><a href='/tag/python/'>python: <small>2</small></a></li>

	<li><a href='/tag/recombination/'>recombination: <small>1</small></a></li>

	<li><a href='/tag/simulation/'>simulation: <small>1</small></a></li>

	<li><a href='/tag/st2/'>st2: <small>1</small></a></li>

	<li><a href='/tag/twitter%20bootstrap/'>twitter bootstrap: <small>1</small></a></li>


				</ul>
			</li>			
			<li class="dropdown">
				<a href="#" class="dropdown-toggle" data-toggle="dropdown" 
				title="Online avatars">
				  <!-- Contact -->
				  <i class="icon-user"></i>
				  <b class="caret"></b>
				</a>
				<ul class="dropdown-menu">
				<!-- http://twitter.github.com/bootstrap/javascript.html#dropdowns -->
					
	
	<li><a href="http://www.yoavram.com/">Home <i class="icon-home"></i></a></li>
	
	
	<li><a href="mailto:yoavram@gmail.com">Email <i class="icon-envelope"></i></a></li>
	
	
	<li><a href="http://www.mendeley.com/profiles/yoav-ram">Mendeley</a></li>
	
	
	<li><a href="http://scholar.google.com/citations?user=RIFmJvYAAAAJ">Scholar Citations</a></li>
	
	
	<li><a href="http://figshare.com/authors/Yoav_Ram/99206">FigShare</a></li>
	
	
	<li><a href="http://twitter.com/yoavram">Twitter <i class="glyphicon-twitter-t"></i></a></li>
	
	
	<li><a href="http://www.linkedin.com/in/yoav-ram">Linkedin <i class="glyphicon-linked-in"></i></a></li>
	
	
	<li><a href="https://github.com/yoavram">GitHub <i class="glyphicon-github"></i></a></li>
	
	
	<li><a href="https://bitbucket.org/yoavram">Bitbucket</a></li>
	
	
	<li><a href="http://code.google.com/u/yoav-ram/">Google Code</a></li>
		
				
				</ul>
			</li>
			<li>
				<a href="http://www.yoavram.com/" 
				title="About me">
					<i class="icon-question-sign"></i>
					&nbsp;&nbsp;
				</a>
			</li>
			<li>
				<a href="http://feeds.feedburner.com/yoavram/msb-recent" rel="alternate" type="application/rss+xml"
				title="Subscribe in a reader">
					<i class="glyphicon-rss"></i>
					&nbsp;&nbsp;				
				</a>
			</li>
			<li>
				<a href="http://feedburner.google.com/fb/a/mailverify?uri=yoavram/msb-recent"
				title="Subscribe by Email">
					<i class="icon-envelope"></i>
					&nbsp;&nbsp;				
				</a>
			</li>		
			</ul>
      </div>
 
    </div>
  </div>
</div>
<!-- end Navbar -->
    
<!-- HEADER -->

<div class="row-fluid">
	<div class="span8 offset1">
	<header class="masthead">
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/mean-fitness-at-the-mutation-selection-balance/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next">
		<a href='/convergence-of-mean-fitness/'>Newer &rarr;</a>
		
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#bottom">To bottom &darr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

	<h1>Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000)</h1>
	
<ul class="breadcrumb" style="background-color: #fff;padding: 0px 0;margin: 0 0 10px;">
	<li class="active"><i class="icon-time"></i> 16 October 2012 at 13:22</li>
	
	<li><span class="divider">/</span>
	<i class="icon-star"></i> <a href='http://blog.yoavram.com/tenaillon-et-al-2000/'>Permalink</a></li>
	
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/asexual%20populations/">asexual populations</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/evolution%20of%20the%20mutation%20rate/">evolution of the mutation rate</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/mutators/">mutators</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/recombination/">recombination</a></li>
	
	<li><span class="divider">/</span>
		<i class="icon-tag"></i> <a href="/tag/simulation/">simulation</a></li>
	
	
	
	<li><span class="divider">/</span>
	
	<!-- sharethis buttons http://sharethis.com/publishers/get-sharing-tools -->
	<span class='st_email' ></span>
	<span class='st_twitter' ></span>
	<span class='st_facebook' ></span>
	<span class='st_googleplus' ></span>
	<span class='st_reddit' ></span>
	<span class='st_linkedin' ></span>
	<span class='st_stumbleupon' ></span>
	<span class='st_delicious' ></span>
	<span class='st_citeulike' ></span>
	<span class='st_connotea' ></span>

	</li>
	</ul>

	</header>
	</div>
</div>


<!-- BODY -->	
<div class="row-fluid">
	<div class="span8 offset1">

		

<div class="post">
	
    <ul>
<li><a href="#overview">Overview</a></li>
<li><a href="#model">Model</a><ul>
<li><a href="#genome">Genome</a></li>
<li><a href="#recombination">Recombination</a></li>
<li><a href="#parameters">Parameters</a></li>
</ul></li>
<li><a href="#stats">Stats</a></li>
<li><a href="#results">Results</a><ul>
<li><a href="#recombination-reduces-mutator-fixation">Recombination reduces mutator fixation</a></li>
<li><a href="#recombination-barriers">Recombination barriers</a></li>
<li><a href="#invasion-to-a-moderate-mutator-background">Invasion to a moderate mutator background</a></li>
<li><a href="#recombination-accelarates-adaptation">Recombination accelarates adaptation</a><ul>
<li><a href="#population-size">Population size</a></li>
<li><a href="#beneficial-mutation-rate">Beneficial mutation rate</a></li>
<li><a href="#selection-coefficient-of-beneficial-mutations">Selection coefficient of beneficial mutations</a></li>
</ul></li>
</ul></li>
<li><a href="#discussion">Discussion</a></li>
<li><a href="#references">References</a></li>
</ul>
<a name='TOC'></a>
<h2 id="overview"><a href="#TOC">Overview</a></h2>
<p>This post is mostly a technical summary of the paper by <strong>Tenaillon, Le Nagard, Godelle and Taddei <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/"><span class="citation">Tenaillon et al. (2000)</span></a></strong>. I wrote the summary because I use it as a baseline for my own research, which involves the evolution of stress-induced mutators [@Ram2012a].</p>
<p>The hypothesis the paper deals with is:</p>
<blockquote>
<p><em>The existence of genetic exchanges could modify the dynamics of adaptation and limits the success of mutator alleles<sup><a href="#fn1" class="footnoteRef" id="fnref1">1</a></sup></em>.</p>
</blockquote>
<p>It is stated that previous models <span class="citation">(Leigh 1973; Johnson 1999)</span> have considered mutators in a sexual population, but didn't consider the case of adaptation with <strong>multiple beneficial mutations</strong> - which can involve different <em>Hill-Robertosn effects</em> <span class="citation">(Hill and Robertson 1966)</span>.</p>
<p>This hypothesis was tested using simulations. The main conclusion, in comparison to a previous work in which the authors studies the evolution of mutators <span class="citation">(Tenaillon et al. 1999)</span>:</p>
<blockquote>
<p><em>Rare genetic exchanges have a dramatic (i.e. negative) effect on the fixation of mutators.</em></p>
</blockquote>
<p>In addition, the paper presents some evidence on <em>why</em> genetic exchanges (recombination) have such an effect of the evolution of mutators.</p>
<h2 id="model"><a href="#TOC">Model</a></h2>
<p>The model is an extension of the density-based model<sup><a href="#fn2" class="footnoteRef" id="fnref2">2</a></sup> used in <span class="citation">(Tenaillon et al. 1999)</span>. The extension is the addition of genetic exchanges (&quot;bacterial recombination&quot;). Both <strong>conjugation</strong> and <strong>transformation</strong> were used, but only the results of transformation are shown.</p>
<p>The model is built using:</p>
<ul>
<li>Finite populations - genetic drift, clonal interference</li>
<li>Directional selection - no epistasis, no changing environments</li>
<li>Several mutations with various fitness effects</li>
<li>Colonization of a new environment by a single cell - adaptive evolution scenario</li>
</ul>
<h3 id="genome"><a href="#TOC">Genome</a></h3>
<p>The basic model of the genome (in parentheses are the values used in my own work for comparison):</p>
<ul>
<li>1 mutator locus (1)</li>
<li>6 loci for beneficial mutations (4+)
<ul>
<li>with additive effect on fitness: 0.06, 0.04, 0.04, 0.03, 0.03, 0.03 - mean 0.03833 (0.1)</li>
<li>all loci treated independently, only one locus will be involved in the process at one time (?)</li>
</ul></li>
<li>1,000 loci for deleterious mutations (+-1,000)
<ul>
<li>with additive effect on fitness: 0.05 (0.1)</li>
</ul></li>
</ul>
<h3 id="recombination"><a href="#TOC">Recombination</a></h3>
<ul>
<li>Drawing the number of recieved genes from a <em>Poisson</em> distribution</li>
<li>Randomly choose the sites, according to the number of sites drawn</li>
<li>Choosing a random allele for each site based on the allele frequency in the population</li>
<li>Constant gene transfer rate</li>
</ul>
<h3 id="parameters"><a href="#TOC">Parameters</a></h3>
<p>Rates are modified from the original to units of <em>events per genome per generation</em>. (in parentheses are the rates used in my own work for comparison):</p>
<ul>
<li>Mutators increase rates: 100-fold (2-10)</li>
<li>Beneficial mutations: 6x10<sup>-8</sup> based on 10<sup>-8</sup> per mutation (1.2x10<sup>-5</sup> - 3x10<sup>-5</sup> - x200)</li>
<li>Deleterious mutations: 10<sup>-4</sup> <span class="citation">(Kibota and Lynch 1996)</span> (3x10<sup>-3</sup> - 2.97x10<sup>-3</sup> <span class="citation">(Drake 1991)</span>- x30)</li>
<li>Lethal mutations: 10<sup>-5</sup> (0)</li>
<li>Non-mutator to mutator: 5x10<sup>-6</sup> (0)</li>
<li>Mutator to non-mutator: 100x5x10<sup>-10</sup> = 5x10<sup>-8</sup> (0)</li>
<li>Genetic exchanges: 0, 0.00001, 0.0001, 0.001, 0.01 and 0.1 (0, 0.015 or 0.75)
<ul>
<li>For comparison, <em>E. coli</em> do ~5x10<sup>-5</sup> and <em>H. pylori</em> ~3x10<sup>-3</sup> <span class="citation">(Milkman and Bridges 1990; Falush et al. 2001)</span>.</li>
</ul></li>
<li>Population size: mostly 10<sup>9</sup>, also checked 10<sup>5</sup>-10<sup>10</sup> (10<sup>5</sup>).</li>
</ul>
<h2 id="stats"><a href="#TOC">Stats</a></h2>
<ul>
<li>Time of adaptation: no. generations for 95% of population to carry all beneficial mutations</li>
<li>Mutator frequency: mean frequency at end of adaptation process over 1,000 replicates</li>
<li>Mutator contribution: when mutator frequency &lt; 50% but at least one mutation that appeared in a mutator is present in &gt;50% of population</li>
<li>Mutator fixation: frequency &gt;50%</li>
</ul>
<h2 id="results"><a href="#TOC">Results</a></h2>
<h3 id="recombination-reduces-mutator-fixation"><a href="#TOC">Recombination reduces mutator fixation</a></h3>
<p>When the rate of genetic exchanges increases, the fixation probability of mutators decreases. <em>Figure 1A</em> shows this very clearly for a 100-fold mutator:</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F1/"><img src="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/bin/pq1800633001.jpg" alt="Figure 1" /></a></p>
<blockquote>
<p>Figure 1 from <span class="citation">Tenaillon et al. (2000)</span>. A) Fixation probability of a 100-fold fold mutator, population size 10^-9. B) Adaptation times of populations with: circle - low mutation rate, square - high mutation rate, triangle - mixed populations.</p>
</blockquote>
<p>However, the <em>B panel</em> shows that the adaptation time of a mutator population is still shorter than that of a non-mutator population, even when mutators are loosing. This is due to the contribution of mutators to adaptation - in a recombining population mutators are <strong>&quot;altruistic&quot;</strong> alleles because they give away the beneficial mutations they generate. This causes them to lose their adaptive advantage and therefore their fixation probability drops, but it still increases the ability of the whole population to adapt.</p>
<p>Note that the rates of recombination in the figure are <em>per gene</em>. Converted to <em>per genome</em> they would be: 0, 10<sup>-5</sup>, 10<sup>-4</sup>, 10<sup>-3</sup>, 10<sup>-2</sup>, 10<sup>-1</sup>, or in decimal representation: 0, 0.00001, 0.0001, 0.001, 0.01, 0.1. Another useful presentation is the <em>ratio of recombination to mutation</em>, using the highest genomic mutation rate (10<sup>-4</sup>): 0, 0.1, 1, 10, 100, 1000, or compared to the 100-fold mutator: 0, 0.001, 0.01, 0.1, 1, 10.</p>
<p>So we see that as soon as the <em>recombination to mutation ratio</em> is 0.1 or lower (4<sup>th</sup> column from the left in <em>Fig. 1A</em>, if we consider the 100-fold increae), the fixation probability of mutators is ~50% or more. The conclusion here could be that mutators can evolve in a recombining population as long as recombination in not more than 10-fold stronger than mutation</p>
<p>Of course this <em>10-fold</em> estimate is highly dependent on the other paramters: the population size, the selection coefficients, the beneficial mutation rate, not to mention parameters that are not dealt with here such as epistasis.</p>
<h3 id="recombination-barriers"><a href="#TOC">Recombination barriers</a></h3>
<p>A recombination barrier, in the context of these simulations, prevents recombination between mutator and non-mutator individuals, limiting recombination events to within the sub-populations.</p>
<p>This is useful here because it allows to test if the source of the effect of recombination on the fixation of mutators is: <strong>1) breaking the linkage</strong> of the mutator with the beneficial mutations it generates, or <strong>2) competing adaptive strategy</strong>.</p>
<p>The 1<sup>st</sup> effect is usually called the <strong>hitch-hiking effect</strong>, or <strong>selective sweeps</strong> <span class="citation">(Charlesworth 2007)</span>: mutators can &quot;hitch-hike&quot; with the beneficial mutations they generate to high frequencies as long as the LD (linkage disequilibrium) between them is intact. recombination tends to break this LD, robbing the mutators from their precious beneficial mutations. Even worse, recombination doesn't only take away the beneifical mutations, it even gives them to the mutators' &quot;rivals&quot; - the competing non-mutator alleles.</p>
<p>The 2<sup>nd</sup> effect is general - <em>competing adadptive strategy</em>. How is recombination an <em>adaptive strategy</em>?</p>
<p>Asexual populations are limited by <strong>clonal interference</strong> <span class="citation">(Gerrish and Lenski 1998; Martens and Hallatschek 2011)</span>. The following figure illustrates the idea:</p>
<p><a href="http://en.wikipedia.org/wiki/Clonal_interference"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Evolsex-dia2a.svg/500px-Evolsex-dia2a.svg.png" alt="Clonal Interference" /></a></p>
<blockquote>
<p>Clonal interference diagram from <a href="http://en.wikipedia.org/wiki/File:Evolsex-dia2a.svg">Wikipedia</a>. This diagram illustrates how sex might create novel genotypes more rapidly. Two beneficial mutations A and B occur at random. The two mutations are recombined rapidly in a sexual population (top), but in an asexual population (bottom) the two mutations must independently arise because of clonal interference. The effect of recombination and sex on the adaptation rate via the reduction of <em>clonal interference</em> is also called the <em>Fisher-Muller effect</em>.</p>
</blockquote>
<p>If the reduction in fixation probability is due to the 1<sup>st</sup> effect - separation of the mutators from the beneficial mutations they generate - then <em>recombination barriers</em> are expected to revert the outcome, at least partially, to that of asexual populations.</p>
<p>However, the results showed that <em>recombination barriers</em> didn't recover the fixation probability of mutators to their levels without recombination. No figure presents the data, but the text provides some information: 7% fixation instead of 60% with beneficial mutation rate 10<sup>-8</sup> and 0.3% instead of 16% with a rate of 10<sup>-9</sup>. The conclusion is therefore that the 1<sup>st</sup> effect is not enough to explain the reduction in the mutators adaptive advantage in the presence of recombination.</p>
<h3 id="invasion-to-a-moderate-mutator-background"><a href="#TOC">Invasion to a moderate mutator background</a></h3>
<p>To check that the 2<sup>nd</sup> effect is the determining effect, a 100-fold mutator invaded another mutator (instead of a non-mutator). The results are in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F2/">Figure 2</a> - in general the fixation probability of a 100-fold mutator drops rapidly when competing with a 10-fold mutator and even a 5-fold mutator.</p>
<p>This illustrates that competing adaptive strategies can reduce the fixation of mutators, and therefore that is is plausible that this is why recombination reduces the fixation of mutators.</p>
<h3 id="recombination-accelarates-adaptation"><a href="#TOC">Recombination accelarates adaptation</a></h3>
<p>Setting the recombination rate at 10<sup>-4</sup> gene conversions per individual per generations, the effect of other parameters was investigated:</p>
<h4 id="population-size"><a href="#TOC">Population size</a></h4>
<p>In asexuals, increasing the population size increases the fixation rate of mutators <span class="citation">(Tenaillon et al. 1999)</span>.</p>
<p>In recombining populations, this is not the case. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F3/">Figure 3A</a> shows that for a population size &lt;10<sup>6</sup>, recombination has a very small effect on the fixation of a 100-fold mutator. However, the trends of the fixation probability are completely different for population size &gt;10<sup>-6</sup>. In asexual populations it increases (rapidly after 10<sup>7</sup>), but in recombining populations it decreases and then increases a bit again at 10<sup>9</sup>, but not much, never surprassing 0.2.</p>
<h4 id="beneficial-mutation-rate"><a href="#TOC">Beneficial mutation rate</a></h4>
<p>The lower the beneficial mutation rate, the lower the fixation probability of mutators. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F4/">Figure 4</a> shows that the fixation probability of mutators starts to increase at a beneficial mutation rate of 10<sup>-9</sup> for asexual populations, but only at 10<sup>-8</sup> for sexual populations.</p>
<h4 id="selection-coefficient-of-beneficial-mutations"><a href="#TOC">Selection coefficient of beneficial mutations</a></h4>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F5/">Figure 5</a> shows that the negative effect of recombination of the fixation of mutators decreases when the selection coefficient of beneficial mutations increases.</p>
<p>For a selection coefficient of 0.01 <span class="citation">(Kibota and Lynch 1996)</span> recombination reduced the fixation probability of mutators 17-fold.</p>
<p>A selection coefficient of 0.1, used in my own work [@Ram2012a], falls between the two right most points of <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27047/figure/F5/">Figure 5</a> - 2.5 and 3 - and the fixation probability of a 100-fold mutator allele drops ~10-fold due to recombination at a rate of 10<sup>-4</sup> per gene per generation in a population with size 10<sup>9</sup>.</p>
<h2 id="discussion"><a href="#TOC">Discussion</a></h2>
<p>This paper is <em>packed</em> with interesting results. The effect of recombination on mutator evolution has not recieved alot of treatment using simulation (but see <span class="citation">Levin and Cornejo (2009)</span> and <span class="citation">Sloan and Panjeti (2010)</span> on which I hope to write some other time). So this is really interesting to someone interesed in the evolution of the mutation rate, even 12 years later.</p>
<p>This work shows that indeed recombination has an effect on mutator evolution, and that this effect is highly sensitive - it is (I'm concentrating on when the effect is weak because that's when mutators are still going to evolve):</p>
<ul>
<li>Weak in small populations (&lt;10<sup>7</sup>-10<sup>8</sup>)</li>
<li>Weak for high beneficial mutation rates (&gt;10<sup>-8</sup>-10<sup>-7</sup>)</li>
<li>Weaker for strong/intermediate selection</li>
</ul>
<p>It seems that in general the effect of recombination on mutator evolution (that is, on the evolution of high mutation rates) is weak when the adaptation rate of the population is low. In turn, the adaptation rate is a function of the <strong>supply of beneficial mutations</strong> which is a function of population size and beneficial mutation rate, and of <strong>fixation time of beneficial mutations</strong> which is a function of population size and selection strength.</p>
<p>Selection of mutators is <strong>second-order selection</strong>, in the sense that it indirectly affecting mutators. Because their results imply that second-order selection of mutators is very sensitive to the system parameters, <em>Tenaillon et al.</em> propose that it may be unable to optimize the rates of mutation - in the sense that mutators will not be selected even when increasing the mutation rate will be beneficial to the population, in terms of fitness and adaptation rate. This can be resolved by <strong>transient hypermutation</strong>:</p>
<blockquote>
<p>Sex would therefore help bacteria approach an optimized mutation rate strategy composed of shifts towards high mutation rates in phases of adaptation under strong selective pressure and rapid recovery of low mutation rate once adaptation is achieved.</p>
</blockquote>
<p>This view was also proposed by Ninio <span class="citation">(1991)</span> and expanded in a review from the same lab with the attractive title <em>The Rise and Fall of Mutator Bacteria</em> <span class="citation">(Giraud et al. 2001)</span>.</p>
<p>I'm not sure if and how these results would change if a weaker mutator (10-fold mutation rate increase?) was used instead of the strong mutator (100-fold). For example, a paper from the same lab <span class="citation">(Taddei et al. 1997)</span> showed that a 10-fold mutator is more successful in invading finite populations (see <a href="http://www.nature.com/nature/journal/v387/n6634/fig_tab/387700a0_F3.html">Figure 3A</a>).</p>
<p>I learned alot from this paper (and it predecessors <span class="citation">(Taddei et al. 1997; Tenaillon et al. 1999)</span>) about how to think of evolutionary simulations. And of course the science itself - the diverse network of interactions and effects between selection, drift, mutation and recombination.</p>
<h2 id="references"><a href="#TOC">References</a></h2>
<p>Charlesworth, Brian. 2007. &#8220;A hitch-hiking guide to the genome: a commentary on &#8217;The hitch-hiking effect of a favourable gene&#8217; by John Maynard Smith and John Haigh.&#8221; <em>Genetical Research</em> 89 (dec): 389&#8211;90. doi:10.1017/S0016672308009580. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18976526" title="http://www.ncbi.nlm.nih.gov/pubmed/18976526">http://www.ncbi.nlm.nih.gov/pubmed/18976526</a>.</p>
<p>Drake, John W. 1991. &#8220;A constant rate of spontaneous mutation in DNA-based microbes.&#8221; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 88 (aug): 7160&#8211;4. doi:10.1073/pnas.88.16.7160. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=52253</a>.</p>
<p>Falush, Daniel, Christian Kraft, Nancy S. Taylor, Pelayo Correa, James G. Fox, Mark Achtman, and Sebastian Suerbaum. 2001. &#8220;Recombination and mutation during long-term gastric colonization by Helicobacter pylori: estimates of clock rates, recombination size, and minimal age.&#8221; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 98 (dec): 15056&#8211;61. doi:10.1073/pnas.251396098. <a href="http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982" title="http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982">http://www.pnas.org/cgi/content/abstract/98/26/15056 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=64982</a>.</p>
<p>Gerrish, Philip J., and Richard E. Lenski. 1998. &#8220;The fate of competing beneficial mutations in an asexual population.&#8221; <em>Genetica</em> 102-103: 127&#8211;144. doi:10.1023/A:1017067816551. <a href="http://www.springerlink.com/content/np187h5878166310/" title="http://www.springerlink.com/content/np187h5878166310/">http://www.springerlink.com/content/np187h5878166310/</a>.</p>
<p>Giraud, Antoine, Ivan Matic, Olivier Tenaillon, Antonio Clara, Miroslav Radman, Michel Fons, and Fran&#231;ois Taddei. 2001. &#8220;Costs and benefits of high mutation rates: adaptive evolution of bacteria in the mouse gut.&#8221; <em>Science</em> 291 (mar): 2606&#8211;8. doi:10.1126/science.1056421. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11283373" title="http://www.ncbi.nlm.nih.gov/pubmed/11283373">http://www.ncbi.nlm.nih.gov/pubmed/11283373</a>.</p>
<p>Hill, W. G., and Alan Robertson. 1966. &#8220;The effect of linkage on limits to artificial selection.&#8221; <em>Genetical Research</em> 8 (dec): 269&#8211;94. doi:10.1017/S0016672300010156. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156" title="http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156">http://www.ncbi.nlm.nih.gov/pubmed/5980116 http://www.journals.cambridge.org/abstract\_S0016672300010156</a>.</p>
<p>Johnson, Toby. 1999. &#8220;Beneficial mutations, hitchhiking and the evolution of mutation rates in sexual populations.&#8221; <em>Genetics</em> 151 (apr): 1621&#8211;31. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&amp;tool=pmcentrez\&amp;rendertype=abstract" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1460574\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</p>
<p>Kibota, Travis T., and Michael Lynch. 1996. &#8220;Estimate of the genomic mutation rate deleterious to overall fitness in E. coli.&#8221; <em>Nature</em> 381 (jun): 694&#8211;6. doi:10.1038/381694a0. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8649513" title="http://www.ncbi.nlm.nih.gov/pubmed/8649513">http://www.ncbi.nlm.nih.gov/pubmed/8649513</a>.</p>
<p>Leigh, Egbert Giles Jr. 1973. &#8220;The evolution of mutation rates.&#8221; <em>Genetics</em> 73 (apr): 73. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4711556" title="http://www.ncbi.nlm.nih.gov/pubmed/4711556">http://www.ncbi.nlm.nih.gov/pubmed/4711556</a>.</p>
<p>Levin, Bruce R., and Omar E. Cornejo. 2009. &#8220;The population and evolutionary dynamics of homologous gene recombination in bacterial populations.&#8221; <em>PLoS Genetics</em> 5 (aug): 1000601. doi:10.1371/journal.pgen.1000601. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&amp;tool=pmcentrez\&amp;rendertype=abstract" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2717328\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</p>
<p>Martens, Erik A., and Oskar Hallatschek. 2011. &#8220;Interfering Waves of Adaptation Promote Spatial Mixing.&#8221; <em>Genetics</em> 189 (sep): 1045&#8211;1060. doi:10.1534/genetics.111.130112. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21900264" title="http://www.ncbi.nlm.nih.gov/pubmed/21900264">http://www.ncbi.nlm.nih.gov/pubmed/21900264</a>.</p>
<p>Milkman, Roger, and Mellissa McKane Bridges. 1990. &#8220;Molecular evolution of the Escherichia coli chromosome. III. Clonal frames.&#8221; <em>Genetics</em> 126 (nov): 505&#8211;17. <a href="http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208" title="http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208">http://mutage.oxfordjournals.org/cgi/doi/10.1093/mutage/2.2.147 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204208</a>.</p>
<p>Ninio, Jacques. 1991. &#8220;Transient mutators: a semiquantitative analysis of the influence of translation and transcription errors on mutation rates.&#8221; <em>Genetics</em> 129 (nov): 957&#8211;62. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&amp;tool=pmcentrez\&amp;rendertype=abstract" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1204761\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</p>
<p>Sloan, Daniel B., and Vijay G. Panjeti. 2010. &#8220;Evolutionary feedbacks between reproductive mode and mutation rate exacerbate the paradox of sex.&#8221; <em>Evolution</em> 64 (apr): 1129&#8211;35. doi:10.1111/j.1558-5646.2009.00869.x. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19863589" title="http://www.ncbi.nlm.nih.gov/pubmed/19863589">http://www.ncbi.nlm.nih.gov/pubmed/19863589</a>.</p>
<p>Taddei, Fran&#231;ois, Miroslav Radman, John Maynard Smith, Bruno Toupance, Pierre-Henri Gouyon, and Bernard Godelle. 1997. &#8220;Role of mutator alleles in adaptive evolution.&#8221; <em>Nature</em> 387 (jun): 700&#8211;2. doi:10.1038/42696. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9192893" title="http://www.ncbi.nlm.nih.gov/pubmed/9192893">http://www.ncbi.nlm.nih.gov/pubmed/9192893</a>.</p>
<p>Tenaillon, Olivier, Herve Le Nagard, Bernard Godelle, and Fran&#231;ois Taddei. 2000. &#8220;Mutators and sex in bacteria: conflict between adaptive strategies.&#8221; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 97 (sep): 10465&#8211;70. doi:10.1073/pnas.180063397. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10973474" title="http://www.ncbi.nlm.nih.gov/pubmed/10973474">http://www.ncbi.nlm.nih.gov/pubmed/10973474</a>.</p>
<p>Tenaillon, Olivier, Bruno Toupance, Herve Le Nagard, Fran&#231;ois Taddei, and Bernard Godelle. 1999. &#8220;Mutators, Population Size, Adaptive Landscape and the Adaptation of Asexual Populations of Bacteria.&#8221; <em>Genetics</em> 152: 485&#8211;493. <a href="http://www.genetics.org/cgi/content/abstract/152/2/485" title="http://www.genetics.org/cgi/content/abstract/152/2/485">http://www.genetics.org/cgi/content/abstract/152/2/485</a>.</p>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Alleles that increase the global mutation rate. For example, a mutant allele of the <em>mutS</em> gene can reduce the efficiency of DNA repair, allowing more mutations to slip though the repair mechanisms.<a href="#fnref1">&#8617;</a></p></li>
<li id="fn2"><p>A density-based model is one where the <em>number</em> of individuals in each class is monitored and modified from one generation to the next. This is in contrast to frequency-dependent models in which the frequency of individuals in each class is monitored.<a href="#fnref2">&#8617;</a></p></li>
</ol>
</div>
	
	<a name="bottom"></a>
	
	
	<ul class="pager">
	  
	  <li class="previous">
		<a href='/mean-fitness-at-the-mutation-selection-balance/'>&larr; Older</a>
	  
	  </li>
	  
	  
	   
	   <li class="next">
		<a href='/convergence-of-mean-fitness/'>Newer &rarr;</a>
		
	  </li>
	</ul>

	
	<ul class="pager">
	<li class="previous">
		<a href='/'>&larr; Back to blog</a>
	</li>
	<li class="next">
		
		<a href="#">To top &uarr;</a>
		
	</li>
	</ul>
	<!-- end of pagination -->

	
</div>


<div id="disqus_thread"></div>
<script type="text/javascript">

	/* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
	/* see http://help.disqus.com/customer/portal/articles/472098-javascript-configuration-variables */
	var disqus_shortname = 'yoavram'; // required: replace example with your forum shortname
	var disqus_identifier = 'tenaillon-et-al-2000';
	
	/* * * DON'T EDIT BELOW THIS LINE * * */
	(function() {
		var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
		dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
		(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>



		
		
	</div>
	<div class="span3">
	<div class="sidebar">
		
	
	
<div class="well-large">
	<strong>Please cite this article as:</strong>
	<br>
	Ram, Yoav (2012) Summary: &#34;Mutators and sex in bacteria: Conflict between adaptive strategies&#34; (Tenaillon et al. 2000). <em>The Mutation-Selection Blog</em>. 
	<a href='http://blog.yoavram.com/tenaillon-et-al-2000/'>http://blog.yoavram.com/tenaillon-et-al-2000/</a>
</div>

	
		
		</div>
	</div>
	
</div>

<!-- FOOTER -->
<div class="row-fluid">
<div class="span10 offset1">
<footer class="colophon"> 
	
	
	<ul class="quick-links">
	<li> 
		<!--a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">
		<img alt="Creative Commons License" src="http://i.creativecommons.org/l/by-sa/3.0/80x15.png">
		</a-->
		The content on "<span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Text" property="dct:title" rel="dct:type">The Mutation-Selection Blog</span>" by <a xmlns:cc="http://creativecommons.org/ns#" href="http://www.yoavram.com/" property="cc:attributionName" rel="cc:attributionURL">Yoav Ram</a>
		is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a>	license.
		<!-- Creative Commons Attribution-ShareAlike 3.0 Unported License -->
		The rest is licensed separately:
	</li>
	</ul>
	<ul class="quick-links">
	<li>Front</li>
	<li><a href="http://blog.getbootstrap.com/">Twitter Bootstrap</a></li>
	<li><a href="http://glyphicons.com/" title="CC BY-SA 3.0">Glyphicons Free</a><li>
	<li><a href="http://code.google.com/p/google-code-prettify/">Google Code Prettifier</a></li>
	<li><a href="http://ethanschoonover.com/solarized">Solarized</a></li>
	<li><a href="http://mathjax.org/">MathJax</a></li>
	</ul>

	<ul class="quick-links">
	<li>Back</li>
	<li><a href="http://python.org/">Python</a></li>
	<li><a href="http://flask.pocoo.org/">Flask</a></li>
	<li><a href="http://pages.github.com/">GitHub Pages</a></li>	
	<li><a href="http://daringfireball.net/projects/markdown/">Markdown</a></li>
	<li><a href="http://johnmacfarlane.net/pandoc">Pandoc</a></li>
	</ul>
  
	<ul class="quick-links">
	<li>Code</li>
	<li><a href="https://github.com/yoavram/yoavram.github.com">Static site</a></li>
	<li><a href="https://github.com/yoavram/yoavram.github.com/tree/source">Site generator</a></li>
	<li>Logo</li>
	<li><a href="http://www.idancohen.com/">Idan Cohen</a></li>
	<li><a href="http://www.dafont.com/sketch-block.font">Sketch Block</a></li>
	<li><a href="http://www.dafont.com/positiv-a.font">Positiv-A</a></li>
	</ul>
	
	<ul class="quick-links">
	<li>This page was last modified on 17 December 2012 at 21:33.</li>
	</ul>
	
	
	<!-- Google Analytics www.google.com/analytics/ -->
	<script type="text/javascript">

	  var _gaq = _gaq || [];
	  _gaq.push(['_setAccount', 'UA-3865698-10']);
	  _gaq.push(['_trackPageview']);

	  (function() {
	    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
	    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
	    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	  })();
	</script>
	
	<a href="/recent.atom" rel="alternate" title="Recent Articles" type="application/atom+xml"></a>
</footer>
</div>
</div>

</body>
</html>